---
layout: talk_reveal3
title: Forecasting strain turnover in influenza and dengue viruses
venue: Kyoto, Japan
event: SMBE Satellite Workshop on Genome Evolution in Pathogen Transmission and Disease
author: Trevor Bedford
---

<!-- Color ramp: ["#5097BA", "#5BA6A7", "#69B091", "#79B77D", "#8BBB6A", "#9EBE5A", "#B2BD4D", "#C4B945", "#D4B13F", "#DFA43B"] -->

<!-- New color ramp: #BD698B, #69BBBD, #A6BD69, #697BBD, #BD9969 -->

<section data-background="#000">
	<h1 class="title">{{ page.title }}</h1>
	<br>
	<p class="title">
	Trevor Bedford and Sidney Bell
	<br>
	{{ page.date | date: "%-d %b %Y" }}
	<br>
	{{ page.event }}
	<br>
	{{ page.venue }}
</section>

<section data-background="#050D16">
	<h3 class="title">Influenza and dengue viruses</h3>
	<div style="vertical-align: bottom; display: table-cell;">
		<img width="48%" src="/images/talks/virion_flu.jpg">
		<img width="48%" src="/images/talks/virion_dengue.jpg">
	</div>
	<div class="citation">
		<a href="https://www.visual-science.com/projects/">Visual Science</a>
	</div>
</section>

<section data-background="#000">
	<h1 class="title">Influenza virus</h1>
</section>

<section>
	<h3>Population turnover is extremely rapid</h3>
	<img class="stretch" data-src="/images/talks/flu_h3n2_tree_beast.png">
</section>

<!-- <section>
	<h3>Clades emerge, die out and take over</h3>
	<img class="stretch" data-src="/images/talks/flu_h3n2_tree_clades.png">
</section>

<section>
	<h3>Clades show rapid turnover</h3>
	<img class="stretch" data-src="/images/talks/prediction_combined_trajectories.png">
</section> -->

<section>
	<h3>Dynamics driven by antigenic drift</h3>
	<img class="stretch" data-src="/images/talks/flu_h3n2_tree.png">
</section>

<section>
	<h3>Necessitates vaccine updates every ~2 years</h3>
	<img class="stretch" data-src="/images/talks/flu_h3n2_tree_vaccines.png">
</section>

<!-- <section>
	<h3>H3N2 vaccine updates occur every ~2 years</h3>
	<img class="stretch" data-src="/images/talks/flu_vaccine_updates_2018_09.png">
</section> -->

<section>
	<h3>Vaccine strain selection by WHO</h3>
	<img class="stretch" data-src="/images/talks/flu_vaccine_schedule.jpg">
</section>

<!---------------------------------->

<section>
	<h1>
		<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span>
	</h1>
	<p>Project to provide a real-time view of the evolving influenza population <br>
		in collaboration with Richard Neher
	</p>
	<div class="citation">
		<a href="/papers/neher-nextflu/">Neher and Bedford. 2015. Bioinformatics.</a>
	</div>
</section>

<section>
	<h3>
		<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span>
		pipeline
	</h3>
	<p>
	<ol>
		<li>Download all recent HA sequences from GISAID</li>
		<li>Filter to remove outliers</li>
		<li>Subsample across time and space</li>
		<li>Align sequences</li>
		<li>Build tree</li>
		<li>Estimate clade frequencies</li>
		<li>Infer antigenic phenotypes</li>
		<li>Export for visualization</li>
	</ol>
</section>



<!----------------------------------------->

<!-- <section data-background="#000">
	<h1 class="title">Antigenic evolution</h1>
	<p class="smaller title"><i>
	with Richard Neher, Rod Daniels, Colin Russell and Boris Shraiman
	</i></p>
</section> -->

<!-- <section>
	<h3>Influenza hemagglutination inhibition (HI) assay</h3>
	<img class="stretch" data-src="/images/talks/hi_assay.png">
</section> -->

<!-- <section>
	<h3>Antigenic phenotype measured by pairwise hemagglutination inhibition (HI) assays</h3>
	<img class="stretch" data-src="/images/talks/hi_cross_reactivity.jpg">
</section> -->

<section>
	<h3>Antigenic phenotype measured by pairwise hemagglutination inhibition (HI) assays</h3>
	<img class="stretch" data-src="/images/talks/hi_table.png">
	<div class="citation">
		<a href="http://www.crick.ac.uk/research/worldwide-influenza-centre/annual-and-interim-reports/">NIMR 2010</a>
	</div>
</section>

<section>
	<h3>Phylogenetic model that ascribes drops in HI titer data to specific branches</h3>
	<img class="stretch" data-src="/images/talks/titers_tree_example.png">
	<div class="citation">
		<a href="http://bedford.io/papers/neher-titers/">Neher et al. 2016. PNAS.</a>
	</div>
</section>
<!--
<section>
	<h3>Identify phylogeny branches associated with drops in HI titer</h3>
	<img class="stretch" data-src="/images/talks/titer_model_with_table.png">
	<div class="citation">
		<a href="http://bedford.io/papers/neher-titers/">Neher et al. 2016. PNAS.</a>
	</div>
</section>

<section>
	<h3>Model can be used to interpolate across tree and predict phenotype of untested viruses</h3>
	<img class="stretch" data-src="/images/talks/titers_tree_model.png">
	<div class="citation">
		<a href="http://bedford.io/papers/neher-titers/">Neher et al. 2016. PNAS.</a>
	</div>
</section>

<section>
	<h3>Model is highly predictive of missing titer values</h3>
	<img class="stretch" data-src="/images/talks/titers_validation.png">
	<div class="citation">
		<a href="http://bedford.io/papers/neher-titers/">Neher et al. 2016. PNAS.</a>
	</div>
</section> -->

<section>
	<h3>Up-to-date analysis publicly available at:</h3>
	<h1>
		<a href="http://nextstrain.org/flu" target="_blank">
			<span class="c1s">n</span><span class="c2s">e</span><span class="c3s">x</span><span class="c4s">t</span><span class="c5s">s</span><span class="c6s">t</span><span class="c7s">r</span><span class="c8s">a</span><span class="c9s">i</span><span class="c10s">n</span><span class="text-light-gray">.org/flu</span>
		</a>
	</h1>
</section>

<!---------------------------------->

<section data-background="#000">
	<h1 class="title">Current diversity</h1>
</section>

<section>
	<h3>H3N2 diversity as of Sep 2018</h3>
	<img class="stretch" data-src="/images/talks/nextflu_sep2018_h3n2_tree_clades.png">
</section>

<section>
	<h3>Clade dynamics show recent success of A1b and A2 viruses</h3>
	<img class="stretch" data-src="/images/talks/nextflu_sep2018_h3n2_freqs.png">
</section>

<section>
	<h3>Clade A1b has drifted viruses</h3>
	<img class="stretch" data-src="/images/talks/nextflu_sep2018_h3n2_tree_titers.png">
</section>

<section>
	<h3>Clade A2 appears driven by reassortment event</h3>
	<img class="stretch" data-src="/images/talks/nextflu_sep2018_h3n2_tangletree_lbi.png">
</section>

<section>
	<h2>Clade success generally has a retrospective narrative</h2>
</section>

<!---------------------------------->

<section data-background="#000">
	<h1 class="title">Forecasting strain turnover</h1>
	<p class="smaller title"><i>
	with John Huddleston and Richard Neher
	</i></p>
</section>

<section data-background="#000">
	<div class="left" style="float:left; width:54%;">
		<h3 class="title">
			<br>&nbsp;<br>&nbsp;<br>
			<p><q>"The future is here, it's just not evenly distributed yet"</q> <br> &mdash; William Gibson</p>
		</h3>
	</div>
	<div class="right" style="float:right; width:43%;">
		<img data-src="/images/talks/neuromancer.jpg">
	</div>
</section>

<section>
	<img class="stretch" data-src="/images/talks/music_industry_v2.png">
</section>

<section>
	<h3>Influenza population turnover</h3>
	<img class="stretch" data-src="/images/talks/flu_turnover.png">
</section>

<section>
	<h3>Seek to explain change in clade frequencies over 1 year</h3>
	<img class="stretch" data-src="/images/talks/prediction_clade_frequencies.png">
</section>

<section>
	<h3>Fitness models can project clade frequencies</h3>
	<p>Clade frequencies $X$ derive from the fitnesses $f$ and frequencies $x$ of constituent viruses, such that</p>
	<p>$$\hat{X}_v(t+\Delta t) = \sum_{i:v} x_i(t) \, \mathrm{exp}(f_i \, \Delta t)$$</p>
	<p>This captures clonal interference between competing lineages</p>
	<img class="stretch" data-src="/images/talks/clonal_interferance.jpg">
	<div class="citation">
		<a href="http://www.nature.com/nature/journal/v507/n7490/full/nature13087.html">&#321;uksza and L&auml;ssig 2014. Nature.</a>
	</div>
</section>

<section>
	<h3>Fitness estimated from viral attributes</h3>
	<p>The fitness $f$ of virus $i$ is estimated as</p>
	<p>$$\hat{f}_i = \beta^\mathrm{A} \, f_i^\mathrm{A} + \beta^\mathrm{B} \, f_i^\mathrm{B} + \ldots$$</p>
	<p>where <i>A</i>, <i>B</i>, etc... are different standardized viral attributes</p>
	<p>We learn $\beta$ coefficients from most recent 12 years of H3N2 evolution</p>
	<p>Optimal $\beta$ coefficients minimize sum of squared errors between observed clade frequencies and
		frequency estimated in a 1-year look ahead</p>
	<p>Fit to a subset of non-nested clades</p>
</section>

<section>
	<h3>Fitness predictors used alone show moderate success</h3>
	<table>
		<thead>
			<tr class="smaller">
				<th width="60%">Model</th>
				<th width="12%">$\beta$ coefficient</th>
				<th width="12%">Growth correlation</th>
				<th width="12%">Growth accuracy</th>
			</tr>
		</thead>
		<tbody>
			<tr>
				<td>Antigenic drift based on HI model (<i>cTiterSub</i>)</td>
				<td>0.33</td>
				<td>0.29</td>
				<td>69%</td>
			</tr>
			<tr>
				<td>Cross-immunity based on epitope mutations (<i>ep_x</i>)</td>
				<td>0.50</td>
				<td>0.18</td>
				<td>60%</td>
			</tr>
			<tr>
				<td>Protein function based on deep mutational scanning (<i>dms</i>)</td>
				<td>0.20</td>
				<td>0.01</td>
				<td>57%</td>
			</tr>
			<tr>
				<td>Clade growth based on local branching index (<i>lbi</i>)</td>
				<td>0.39</td>
				<td>0.30</td>
				<td>61%</td>
			</tr>
		</tbody>
	</table>
</section>

<section>
	<h3>Combine predictors into a single model</h3>
	<img class="stretch" data-src="/images/talks/prediction_coefficients_v2.png">
</section>

<section>
	<h3>Prospective trajectories from ensemble model</h3>
	<img class="stretch" data-src="/images/talks/prediction_trajectories_grid_v2.png">
</section>

<section>
	<h3>Growth rate of clades is well predicted</h3>
	<img class="stretch" data-src="/images/talks/prediction_validation_growth_rates_v2.png">
</section>

<section>
	<h3>Prediction of growth vs decline also well predicted</h3>
	<img class="stretch" data-src="/images/talks/prediction_validation_accuracy_v2.png">
</section>

<section>
	<h3>Currently circulating virus clades</h3>
	<img class="stretch" data-src="/images/talks/prediction_2018-11_tree.png">
</section>

<section>
	<h3>Projections of these clades</h3>
	<img class="stretch" data-src="/images/talks/prediction_2018-11_trajectories.png">
</section>

<section>
	<h2>Unlikely to supplant expert predictions, but useful for vaccine candidate selection</h2>
</section>

<!---------------------------------->

<!-- DENGUE -->

<section data-background="#000">
	<h1 class="title">Dengue virus</h1>
</section>

<section>
	<h3>Different tempos of evolution</h3>
	<div class="left" style="float:left;width:50%">
		<p>Dengue (serotype 2)</p>
		<img height="350px" data-src="/images/talks/dengue_nextstrain_tree.png">
	</div>
	<div class="right" style="float:right;width:50%">
		<p>Flu (H3N2)</p>
		<img height="350px" data-src="/images/talks/flu_nextstrain_tree.png">
	</div>
</section>

<!-- SB: Label serotypes in all the tree figures -->
<section>
	<h3>Four (uniform?) serotypes of dengue</h3>
	<img class="stretch" src="/images/talks/dengue_serotype_tree.png">
</section>

<section>
	<h3>Each serotype is genetically diverse</h3>
	<img class="stretch" src="/images/talks/dengue_genotype_tree.png">
</section>

<!-- SB: Add an arrow -->
<section>
	<h3>Sanofi vaccine efficacy varies by genotype</h3>
	<img class="stretch" src="/images/talks/dengue_ve_tree.png">
		<p>DENV4-I associated with adverse events</p>
	<div class="citation">
		<a href="http://www.pnas.org/content/115/36/E8378/">Juraska et al. 2018. PNAS.</a>
	</div>
</section>

<section data-transition="slide-in none-out">
	<h3>Original antigenic sin drives<br>dengue case outcomes</h3>
	<img class="stretch" src="/images/talks/dengue_oas1.png">
</section>

<section data-transition="none-in slide-out">
	<h3>Original antigenic sin drives<br>dengue case outcomes</h3>
	<img class="stretch" src="/images/talks/dengue_oas2.png">
</section>

<section>
	<h3>DENV antigenic relationships<br>are poorly understood</h3>
	<div class="left" style="float:left;width:40%">
		<img height="520px" data-src="/images/talks/dengue_icons.png">
	</div>
	<div class="right" style="float:right;width:60%">
	<p style="text-align: left;">
	Serotypes are genetically distinct<br><br><br>
	Serotypes are antigenically distinct<br><br><br>
	Clades are genetically distinct<br><br><br>
	<strong>Are clades antigenically distinct?</strong>
	</div>
</section>

<!-- Add antigenic cartography slide as a "subslide" (how?) -->

<section>
	<h1>How does<br>dengue evolve<br>antigenically?</h1>
</section>

<section>
	<h3>Models of antigenic evolution</h3>
	<div class="left" style="float:left;width:50%">
		<p>Interserotype hypothesis</p>
		<img data-src="/images/talks/dengue_interserotype_model.png"  style="border: 4px solid #4477aa;" >
	</div>
	<div class="right" style="float:right;width:50%">
		<p>Full tree hypothesis</p>
		<img data-src="/images/talks/dengue_fulltree_model.png" style="border: 4px solid #aa4455;" >
	</div>
</section>

<section>
	<h3>Dataset</h3>
<img class="stretch" data-src="/images/talks/dengue_titer_heatmap.png">
	<p>PRNT50 titers from monovalent vaccine trials + nonhuman primates</p>
	<div class="citation">
		<a href="http://science.sciencemag.org/cgi/pmidlookup?view=long&pmid=26383952">Katzelnick et al. 2015</a>
	</div>
</section>

<section>
	<h3>Within-serotype variation significantly contributes to dengue antigenic phenotypes</h3>
	<div class="left" style="float:left;width:45%">
		<p style="text-align: center;">Interserotype model</p>
		<img data-src="/images/talks/dengue_titer_interserotype_performance.png" >
	</div>
	<div class="right" style="float:right;width:45%">
		<p style="text-align: center;">Full tree model</p>
		<img data-src="/images/talks/dengue_titer_fulltree_performance.png" >
	</div>
	<div class="citation">
	 <a href="https://www.biorxiv.org/content/early/2018/10/02/432054">Bell et al. 2018. bioRxiv.</a>
	</div>
</section>

<section>
	<h3>Each serotype of dengue contains multiple<br>distinct antigenic phenotypes</h3>
	<img class="stretch" data-src="/images/talks/dengue_antigenic_tree.png">
	<div class="citation">
		<a href="https://www.biorxiv.org/content/early/2018/10/02/432054">Bell et al. 2018. bioRxiv.</a>
	</div>
</section>

<section>
	<h3>Dengue antigenic evolution is ongoing but slow</h3>
	<div class="left" style="float:left;width:45%">
		<p>Dengue</p>
		<img data-src="/images/talks/dengue_antigenic_scatterplot.png">
	</div>
	<div class="right" style="float:right;width:45%">
		<p>Flu (H3N2)</p>
		<img data-src="/images/talks/flu_antigenic_scatterplot.png">
	</div>
</section>

<section>
	<h1>Does antigenic diversity impact dengue population dynamics?</h1>
</section>

<section data-transition="slide-in none-out">
	<h3>Serotypes cycle through populations</h3>
	<img height="300px" width="750px" data-src="/images/talks/dengue_serotype_frequencies.png">
</section>

<section data-transition="none-in slide-out">
	<h3>Genotypes cycle through populations</h3>
	<img height="300px" width="750px" data-src="/images/talks/dengue_genotype_frequencies.png">
</section>

<section>
	<h3>Predict clade growth based on fitness (as before)</h3>
	<br>
	<p>Clade frequencies $X$ derive from the clade fitness $f_i$, such that</p>
	<p>$$\hat{X}_i(t+\Delta t) = X_i(t) \, \mathrm{exp}(f_i \, \Delta t)$$</p>
	<p>Estimate fitness as frequency-weighted antigenic distance*<br>from recently circulating clades</p>
	<p>* estimated from the <span style="color: #4477aa;">interserotype</span> or <span style="color: #aa4455;">full tree</span> antigenic model.</p>
	<div class="citation">
		<a href="http://www.nature.com/nature/journal/v507/n7490/full/nature13087.html">&#321;uksza and L&auml;ssig. 2014. Nature.</a>
	</div>
</section>

<section>
	<h3>Fitness based on antigenic distance from<br>standing population immunity</h3>
	<div class="left" style="float:left;width:15%">
	<p style="text-align: right;">
		Circulating clades:<br><br>
		Population immunity:<br><br>
		Population susceptibility:<br><br>
		Clade growth:</p>
	</div>
	<div class="right" style="float:right;width:85%">
		<img height="520px" data-src="/images/talks/dengue_popimmunity_icons.png">
	</div>
</section>

<!-- Add fitness model formula as "bonus" slide -->

<section data-transition="slide-in none-out">
	<h3>Serotype antigenic relationships<br>drive <strong>sero</strong>type dynamics</h3>
	<img height="400px" data-src="/images/talks/dengue_fitness_serotype_performance.png">
	<div class="citation">
		<a href="https://www.biorxiv.org/content/early/2018/10/02/432054">Bell et al. 2018. bioRxiv.</a>
	</div>
</section>

<section data-transition="none-in slide-out">
	<h3>Serotype antigenic relationships<br>drive <strong>geno</strong>type dynamics</h3>
	<div class="left" style="float:left;width:45%;">
		<p style="text-align: center;">Interserotype model</p>
		<img data-src="/images/talks/dengue_fitness_genotype_interserotype_performance.png" >
	</div>
	<div class="right" style="float:right;width:45%;">
		<p style="text-align: center;">Full tree model</p>
		<img data-src="/images/talks/dengue_fitness_genotype_fulltree_performance.png" >
	</div>
	<div class="citation">
		<a href="https://www.biorxiv.org/content/early/2018/10/02/432054">Bell et al. 2018. bioRxiv.</a>
	</div>
</section>

<section>
	<h3>Antigenic fitness drives clade growth &amp; decline</h3>
	<div class="left" style="float:left;width:60%">
		<img width="550px" data-src="/images/talks/dengue_serotype_frequencies_nolegend.png">
		<img width="550px" data-src="/images/talks/prediction_combined_trajectories.png">
	</div>
	<div class="right" style="float:right;width:40%">
		<p style="text-align: left;">
		<strong>Dengue serotype flux</strong><br>
		62% of variation explained<br>
		5 year windows
		</p>
		<br><br><br><br>
		<p style="text-align:left;">
		<strong>Flu clade turnover</strong><br>
		53% of variation explained<br>
		1 year windows
	</div>
</section>

<section>
	<h3>Conclusions</h3>
	<br>
	<p>Similar models of antigenic evolution are effective<br>for pathogens with very different evolutionary dynamics</p>
	<br>
	<p>Prospective modeling can reveal relative contributions to viral fitness</p>
</section
<!---------------------------------->

<section>
	<h3>Acknowledgements</h3>

	<p class="tiny"><b>Bedford Lab</b>:
		<img width=50 class="img-avatar" src="/images/team/allison-black.jpg"> Alli Black,
		<img width=50 class="img-avatar" src="/images/team/sidney-bell.jpg"> Sidney Bell,
		<img width=50 class="img-avatar" src="/images/team/john-huddleston.jpg"> John Huddleston,
		<img width=50 class="img-avatar" src="/images/team/barney-potter.jpg"> Barney Potter,
		<br>
		<img width=50 class="img-avatar" src="/images/team/james-hadfield.jpg"> James Hadfield,
		<img width=50 class="img-avatar" src="/images/team/louise-moncla.jpg"> Louise Moncla,
		<img width=50 class="img-avatar" src="/images/team/thomas-sibley.jpg"> Tom Sibley,
		<img width=50 class="img-avatar" src="/images/team/maya-lewinsohn.jpg"> Maya Lewinsohn
		</p>

	<p class="tiny"><b>Influenza</b>: WHO Global Influenza Surveillance Network, GISAID, Richard Neher, John Huddleston,
	Barney Potter, James Hadfield, Rod Daniels, Boris Shraiman, Colin Russell, Andrew Rambaut, Dave Wentworth, Becky Garten, Jackie Katz,
	Marta &#321;uksza, Michael L&auml;ssig, Richard Reeve</p>

	<p class="tiny"><b>Dengue</b>: Leah Katzelnick, Molly O'hainle, Richard Neher, Paul Edlefsen, Michal Juraska</p>

	<div class="pull-right" style="vertical-align: bottom; display: table-cell;">
		<img width="11%" data-src="/images/talks/logo_fred_hutch_vertical.png">
		&nbsp; &nbsp;
		<img width="7%" data-src="/images/talks/logo_pew.png">
		&nbsp; &nbsp;
		<img width="8%" data-src="/images/talks/logo_hhmi.jpg">
		&nbsp; &nbsp;
		<img width="10%" data-src="/images/talks/logo_wellcome_trust.jpg">
		&nbsp; &nbsp;
		<img width="6%" data-src="/images/talks/logo_nih.jpg">
		&nbsp; &nbsp;
		<img width="11%" data-src="/images/talks/logo_osp.jpg">
		&nbsp; &nbsp;
		<img width="9%" data-src="/images/talks/logo_grfp.png">
	</div>
</section>

<section data-background="#000">
	<img width=100 src="/images/talks/giphy_virus.gif">
</section>
