---
layout: talk_reveal3
title: Nextstrain Workshop
venue: CDC
event: AMD Training
author: Trevor Bedford
---

<!-- Total time of 15 minutes. -->

<!-- Outline:
1. Intro (toy transmission model)
2. Applications to a large variety of pathogens and epidemiological questions
 a. Monitoring evolution and spread of new strains: influenza
 b. Epidemic emergence: Zika
 c. Geographic spread: Ebola
 d. Stuttering chains and animal-to-human spillover: MERS
 e. Identification of transmission chains: TB
3. Nextstrain and real-time analysis.
4. Recommendations

-->

<!-- Color ramp: ["#511EA8", "#4432BD", "#3F4BCA", "#4065CF", "#447ECC", "#4C91BF", "#56A0AE", "#63AC9A", "#71B486", "#81BA72", "#94BD62", "#A7BE54", "#BABC4A", "#CBB742", "#D9AE3E", "#E29E39", "#E68935", "#E56E30", "#E14F2A", "#DC2F24"] -->

<section data-background="#000000">
	<h1 class="title">{{ page.title }}</h1>
	<br>
	<p class="title">
	Trevor Bedford (@trvrb)
	<br>
	{{ page.date | date: "%-d %b %Y" }}
	<br>
	{{ page.event }}
	<br>
	{{ page.venue }}
</section>

<!--- Background --->

<section data-background="#000">
	<h1 class="title">Sequencing to reconstruct pathogen spread</h1>
</section>

<section data-transition="slide-in none-out">
	<h3>Epidemic process</h3>
	<img class="stretch" src="/images/talks/infection_tree_1.png">
</section>

<section data-transition="none">
	<h3>Sample some individuals</h3>
	<img class="stretch" src="/images/talks/infection_tree_2.png">
</section>

<section data-transition="none">
	<h3>Sequence and determine phylogeny</h3>
	<img class="stretch" src="/images/talks/infection_tree_3b.png">
</section>

<section data-transition="none-in slide-out">
	<h3>Sequence and determine phylogeny</h3>
	<img class="stretch" src="/images/talks/infection_tree_4b.png">
</section>

<section style="text-align: left;">
	<h3>Pathogen genomes may reveal:</h3>
	<p>
	<ul>
		<li>Evolution of new adaptive variants</li>
		<li>Epidemic origins</li>
		<li>Patterns of geographic spread</li>
		<li>Animal-to-human spillover</li>
		<li>Transmission chains</li>
	</ul>
	</p>
</section>

<!--- Applications --->

<section>
	<h3><b>Influenza:</b> Forecasting spread of new variants for vaccine strain selection</h3>
	<img class="stretch" src="/images/talks/nextflu_apr2018_h3n2_tree_titers.png">
	<div class="citation">
		<a href="/papers/bedford-feb-2018-flu-report/">Bedford and Neher. 2018. bioRxiv.</a>
	</div>
</section>

<section>
	<h3><b>Zika:</b> Uncovering origins of the epidemic in the Americas</h3>
	<img class="stretch" src="/images/talks/zika_zibra_tree.png">
	<div class="citation">
		<a href="/papers/faria-zika-in-brazil/">Faria et al. 2017. Nature.</a>
	</div>
</section>

<section>
	<h3><b>Ebola:</b> Revealing spatial spread and persistence in West Africa</h3>
	<img class="stretch" src="/images/talks/ebola_mig_exploded_tree_wide.png">
	<div class="citation">
		<a href="/papers/dudas-ebola-epidemic-spread/">Dudas et al. 2017. Nature.</a>
	</div>
</section>

<section>
	<h3><b>MERS:</b> Quantifying camel-to-human spillover</h3>
	<img class="stretch" src="/images/talks/mers_tree.png">
	<div class="citation">
		<a href="/papers/dudas-mers-structure/">Dudas et al. 2018. eLife.</a>
	</div>
</section>

<section>
	<h3><b>TB:</b> Tracking individual transmission chains</h3>
	<img class="stretch" src="/images/talks/tb_transmission_tree.png">
	<div class="citation">
		<a href="https://doi.org/10.1093/molbev/msu121">Didelot et al. 2014. Mol Biol Evol.</a>
	</div>
</section>

<!-- Add food borne or perhaps cholera example -->

<!--- Nextstrain --->

<section data-background="#000">
	<h1 class="title">Actionable inferences</h1>
</section>

<section>
	<h3>Genomic analyses are mostly done in a retrospective manner</h3>
	<img class="stretch" data-src="/images/talks/ebola_sampling_rugplots.png">
	<div class="citation">
		Dudas and Rambaut 2016
	</div>
</section>

<section style="text-align: left;">
	<h3>Key challenges to making genomic epidemiology actionable</h3>
	<p>
	<ul>
		<li>Timely analysis and sharing of results critical</li>
		<li>Dissemination must be scalable</li>
		<li>Integrate many data sources</li>
		<li>Results must be easily interpretable and queryable</li>
	</ul>
	</p>
</section>

<section>
	<h1>
		<span class="c1s">N</span><span class="c2s">e</span><span class="c3s">x</span><span class="c4s">t</span><span class="c5s">s</span><span class="c6s">t</span><span class="c7s">r</span><span class="c8s">a</span><span class="c9s">i</span><span class="c10s">n</span>
	</h1>
	<p>Project to conduct real-time molecular epidemiology and evolutionary analysis of emerging epidemics</p>
	<br>
	<p class="smaller"><i>
		with
		<img width=40 class="img-avatar" src="/images/team/richard-neher.jpg"> Richard Neher,
		<img width=40 class="img-avatar" src="/images/team/james-hadfield.jpg"> James Hadfield,
		<img width=40 class="img-avatar" src="/images/team/emma-hodcroft.jpg"> Emma Hodcroft,
		<img width=40 class="img-avatar" src="/images/team/thomas-sibley.jpg"> Tom Sibley,
		<br>
		<img width=40 class="img-avatar" src="/images/team/john-huddleston.jpg"> John Huddleston,
		<img width=40 class="img-avatar" src="/images/team/colin-megill.jpg"> Colin Megill,
		<img width=40 class="img-avatar" src="/images/team/sidney-bell.jpg"> Sidney Bell,
		<img width=40 class="img-avatar" src="/images/team/barney-potter.jpg"> Barney Potter,
		<br>
		<img width=40 class="img-avatar" src="/images/team/charlton-callender.jpg"> Charlton Callender
	</i></p>
</section>

<!-- <section>
	<div class="left" style="float:left;width:63%">
		<h3>Flexible pipelines constructed through command line modules</h3>
		<p>
		<ul>
			<li>Modules called via <code>augur filter</code>, <code>augur tree</code>, <code>augur traits</code>, etc...
			<li>Designed to be composable across pathogen builds</li>
			<li>Defined pipeline, making steps obvious</li>
			<li>Provides dependency graph for fast recomputation</li>
			<li>Pathogen-specific repos give users an obvious foundation to build from</li>
		</ul>
	</div>
	<div class="right" style="float:right;width:35%">
		<img data-src="/images/talks/nextstrain_augur_dag.png">
	</div>
</section> -->

<section style="text-align: left;">
	<h3>Nextstrain is two things</h3>
	<p>
	<ul>
		<li>a bioinformatics toolkit and visualization app, which can be used for a broad range of datasets</li>
		<li>a collection of real-time pathogen analyses kept up-to-date on the website nextstrain.org</i></li>
	</ul>
	</p>
</section>

<section>
	<h3>Nextstrain architecture</h3>
	<p>All code open source at <a class="off" href="https://github.com/nextstrain">github.com/nextstrain</a></p>
	<p>Two central aims: (1) <i>rapid and flexible phylodynamic analysis</i> and <br> (2) <i>interactive visualization</i></p>
	<img class="stretch" src="/images/talks/nextstrain_components.png">
</section>

<section style="text-align: left;">
	<h2><a href="https://github.com/nextstrain/fauna">Fauna</a></h2>
	<p>
		<b>Rethink database of virus and titer data</b>
	</p>
	<p >
	<ul>
		<li>Harmonizes data from different sources</li>
		<li>Integrates different types of data (serology, sequences, case details)</li>
		<li>Provides an interface for downstream analysis</li>
	<ul>
	</p>
</section>

<section style="text-align: left;">
	<h2><a href="https://github.com/nextstrain/augur">Augur</a></h2>
	<p>
		<b>Build scripts to align sequences, build trees and annotate</b>
	</p>
	<p>
	<ul>
		<li>Flexible build scripts to incorporate different viruses and analyses</li>
		<li>Constructs time-resolved phylogenies</li>
		<li>Annotates with geographic transitions and mutation events</li>
	<ul>
	</p>
</section>

<section style="text-align: left;">
	<h3>Example augur pipeline for 1600 Ebola genomes</h3>
	<p>
	<ul>
		<li>Align with MAFFT <i>(34 min)</i></li>
		<li>Build ML tree with RAxML <i>(54 min)</i></li>
		<li>Temporally resolve tree and geographic ancestry with TreeTime <i>(16 min)</i></li>
		<li>Total pipeline <i>(1 hr 46 min)</i></li>
	</ul>
	</p>
</section>

<section>
	<div class="left" style="float:left;width:63%">
		<h3>Pipeline consists of Unix-like command line modules</h3>
		<p>
		<ul>
			<li>Modules called via <code>augur filter</code>, <code>augur tree</code>, <code>augur traits</code>, etc...
			<li>Designed to be composable across pathogen builds</li>
			<li>Uses Snakemake to define a pipeline, making steps obvious</li>
			<li>Provides depedency graph for fast recomputation</li>
			<li>Pathogen-specific repos give users an obvious foundation to build off of</li>
		</ul>
	</div>
	<div class="right" style="float:right;width:35%">
		<img data-src="/images/talks/nextstrain_augur_dag.png">
	</div>
</section>

<section style="text-align: left;">
	<h2><a href="https://github.com/nextstrain/auspice">Auspice</a></h2>
	<p>
		<b>Web visualization of resulting trees</b>
	</p>
	<p>
	<ul>
		<li>Interactive data exploration and filtering</li>
		<li>Framework through React / D3</li>
		<li>Connects phylogeny, geography and genotypes</li>
	<ul>
	</p>
</section>

<section>
	<h2>
		Demo
		<a href="http://nextstrain.org/" target="_blank">
			<span class="c1s">n</span><span class="c2s">e</span><span class="c3s">x</span><span class="c4s">t</span><span class="c5s">s</span><span class="c6s">t</span><span class="c7s">r</span><span class="c8s">a</span><span class="c9s">i</span><span class="c10s">n</span><span class="text-gray">.org</span>
		</a>
		focusing on visualization features
	</h2>
</section>

<section style="text-align: left;">
	<h3>Todays' workshop</h3>
	<p>
	<ul>
		<li>Brief background on phylodynamics</li>
		<li>Focus of running augur and constructing snakefiles</li>
		<li>Running auspice</li>
	</ul>
	</p>
</section>

<!-- Phylogenetics -->

<section data-transition="none">
	<h3>Phylogeny describes evolutionary relationships</h3>
	<img class="stretch" src="/images/talks/phylogenetic_inference_parsimony_1.png">
</section>

<section data-transition="none">
	<h3>Phylogeny is usually a hypothesis based on characteristics of sampled taxa</h3>
	<img class="stretch" src="/images/talks/phylogenetic_inference_parsimony_2.png">
</section>

<section data-transition="none">
	<h3>Phylogeny implies a series of mutational events leading to observed tip states</h3>
	<img class="stretch" src="/images/talks/phylogenetic_inference_parsimony_3.png">
</section>

<section data-background="#4C91BF">
	<h2 class="title">Phylogenetic inference</h2>
</section>

<section>
	<img class="stretch" src="/images/talks/phylogenetic_inference_schematic.png">
</section>

<section>
	<h3>"Data" is generally a sequence alignment</h3>
	<img class="stretch" src="/images/talks/phylogenetic_inference_alignment.png">
</section>

<section>
	<h3>Phylogeny structures site patterns</h3>
	<img class="stretch" src="/images/talks/phylogenetic_inference_site_patterns.png">
	<div class="citation">
		<a href="http://vincebuffalo.org/dancinggenealogies2/">
			Buffalo. 2015
		</a>
	</div>
</section>

<section style="text-align: left;">
	<h3>Tree space is vast</h3>
	<p>
	There are (2<i>n</i>-3)!! rooted trees for <i>n</i> taxa
	<ul>
		<li>3 taxa: 3 trees</li>
		<li>5 taxa: 105 trees</li>
		<li>10 taxa: 34,459,425 trees</li>
		<li>20 taxa: 8.2 &times; 10<sup>21</sup> trees</li>
		<li>50 taxa: 2.8 &times; 10<sup>76</sup> trees</li>
		<li>100 taxa: 3.3 &times; 10<sup>184</sup> trees</li>
	</ul>
	</p>
</section>

<section>
	<h3>Solution space is rugged</h3>
	<img class="stretch" src="/images/talks/phylogenetic_inference_solution_space.png">
</section>

<section style="text-align: left;">
	<h3>Types of phylogenetic inference methods</h3>
	<p>
	<ul>
		<li>Distance-based (neighbor-joining, fast, heuristic)</li>
		<li>Parsimony (fast, "model-free")</li>
		<li>Maximum likelihood (infers model of mutation, accurate, examples: FastTree, RAxML)</li>
		<li>Bayesian (like ML, but requires prior, produces estimates of uncertainty, examples: MrBayes, BEAST)</li>
	</ul>
	</p>
</section>

<section>
	<h3>Inference is a tree topology, branch lengths and ancestral states</h3>
	<img class="stretch" src="/images/talks/phylogenetic_inference_ancestral_state_tree.png">
	<div class="citation">
		<a href="http://nextflu.org/">nextflu.org</a>
	</div>
</section>

<section data-background="#56A0AE">
	<h2 class="title">Molecular clocks and dated phylogenies</h2>
</section>

<section>
	<h3>Mutations tend to accumulate in a clock-like fashion</h3>
	<img class="stretch" src="/images/talks/phylogenetic_inference_clock_flu_tree.png">
</section>

<section>
	<h3>"Root-to-tip" plots show temporal signal</h3>
	<img class="stretch" src="/images/talks/phylogenetic_inference_clock_flu_root_to_tip.png">
</section>

<section>
	<h3>Allows conversion between branch length and time</h3>
	<img class="stretch" src="/images/talks/phylogenetic_inference_clock_flu_timetree.png">
</section>

<section>
	<h3>Dated phylogenies provide real-world context</h3>
	<img class="stretch" src="/images/talks/phylogenetic_inference_clock_zika_timings.png">
	<div class="citation">
		<a href="http://bedford.io/papers/grubaugh-zika-in-florida/">Grubaugh et al. 2017. Nature.</a>
	</div>
</section>

<section data-background="#63AC9A">
	<h2 class="title">Inference of discrete traits</h2>
</section>

<section>
	<h3>"Data" is a phylogeny and tip states</h3>
	<p>States include nucleotides, amino acids, geo locations, hosts, etc...</p>
	<img class="stretch" src="/images/talks/phylogenetic_inference_phylogeo_inference_data.png">
</section>

<section>
	<h3>Model infers transition matrix and ancestral states</h3>
	<img class="stretch" src="/images/talks/phylogenetic_inference_phylogeo_inference_model.png">
</section>

<section>
	<h3>Rare transitions, short branches and many taxa increase confidence</h3>
	<img class="stretch" src="/images/talks/phylogenetic_inference_phylogeo_inference_rare_transitions.png">
</section>

<section data-background="#71B486">
	<h2 class="title">Phylogeography</h2>
</section>

<section>
	<h3>Nesting patterns are informative</h3>
	<img class="stretch" src="/images/talks/phylogenetic_inference_phylogeo_inference_nesting.png">
</section>

<section>
	<h3>Zika phylogeny infers an origin in northeast Brazil</h3>
	<img class="stretch" src="/images/talks/phylogenetic_inference_phylogeo_zika_tree.png">
	<div class="citation">
		<a href="http://bedford.io/papers/faria-zika-in-brazil/">Faria et al. 2017. Nature.</a>
	</div>
</section>

<section data-background="#000">
	<h1 style="font-weight: 500;">
		<a href="http://nextstrain.org/docs" target="_blank">
			<span class="c1s">n</span><span class="c2s">e</span><span class="c3s">x</span><span class="c4s">t</span><span class="c5s">s</span><span class="c6s">t</span><span class="c7s">r</span><span class="c8s">a</span><span class="c9s">i</span><span class="c10s">n</span><span class="text-light-gray">.org/docs</span>
		</a>
	</h1>
</section>

<section>
	<h3>Acknowledgements</h3>

	<p class="tiny"><b>Bedford Lab</b>:
		<img width=50 class="img-avatar" src="/images/team/allison-black.jpg"> Alli Black,
		<img width=50 class="img-avatar" src="/images/team/john-huddleston.jpg"> John Huddleston,
		<img width=50 class="img-avatar" src="/images/team/barney-potter.jpg"> Barney Potter,
		<img width=50 class="img-avatar" src="/images/team/james-hadfield.jpg"> James Hadfield,
		<br>
		<img width=50 class="img-avatar" src="/images/team/louise-moncla.jpg"> Louise Moncla,
		<img width=50 class="img-avatar" src="/images/team/thomas-sibley.jpg"> Tom Sibley,
		<img width=50 class="img-avatar" src="/images/team/maya-lewinsohn.jpg"> Maya Lewinsohn,
		<img width=50 class="img-avatar" src="/images/team/katie-kistler.jpg"> Katie Kistler
	</p>

	<p class="tiny"><b>Nextstrain</b>: Richard Neher, James Hadfield, Emma Hodcroft, Tom Sibley, John Huddleston,
	Sidney Bell, Barney Potter, Colin Megill, Charlton Callender</p>

	<div class="pull-right" style="vertical-align: bottom; display: table-cell;">
		<img width="15%" data-src="/images/talks/logo_fred_hutch_vertical.png">
		&nbsp; &nbsp;
		<img width="11%" data-src="/images/talks/logo_pew.png">
		&nbsp; &nbsp;
		<img width="10%" data-src="/images/talks/logo_hhmi.jpg">
		&nbsp; &nbsp;
		<img width="12%" data-src="/images/talks/logo_wellcome_trust.jpg">
		&nbsp; &nbsp;
		<img width="10%" data-src="/images/talks/logo_nih.jpg">
		&nbsp; &nbsp;
		<img width="15%" data-src="/images/talks/logo_osp.jpg">
	</div>
</section>
