---
layout: talk_reveal3
title: Genomic surveillance of pathogen evolution and antimicrobial resistance
venue: Institute for Systems Biology
event: AMR Brainstorming Session
author: Trevor Bedford
draft: true
---

<!-- Total time of 15 minutes. -->

<!-- Outline:
1. Intro (toy transmission model)
2. Applications to a large variety of pathogens and epidemiological questions
 a. Monitoring evolution and spread of new strains: influenza
 b. Epidemic emergence: Zika
 c. Geographic spread: Ebola
 d. Stuttering chains and animal-to-human spillover: MERS
 e. Identification of transmission chains: TB
3. Nextstrain and real-time analysis.
4. Recommendations

-->

<!-- Color ramp: ["#511EA8", "#4432BD", "#3F4BCA", "#4065CF", "#447ECC", "#4C91BF", "#56A0AE", "#63AC9A", "#71B486", "#81BA72", "#94BD62", "#A7BE54", "#BABC4A", "#CBB742", "#D9AE3E", "#E29E39", "#E68935", "#E56E30", "#E14F2A", "#DC2F24"] -->

<section data-background-iframe="/talks/viz/gradient/">
	<h1 class="title">{{ page.title }}</h1>
	<br>
	<p class="title">
	Trevor Bedford (@trvrb)
	<br>
	{{ page.date | date: "%-d %b %Y" }}
	<br>
	{{ page.event }}
	<br>
	{{ page.venue }}
</section>

<!--- Background --->

<section data-background="#000">
	<h1 class="title">Sequencing to reconstruct pathogen spread</h1>
</section>

<section data-transition="slide-in none-out">
	<h3>Epidemic process</h3>
	<img class="stretch" src="/images/talks/infection_tree_1.png">
</section>

<section data-transition="none">
	<h3>Sample some individuals</h3>
	<img class="stretch" src="/images/talks/infection_tree_2.png">
</section>

<section data-transition="none">
	<h3>Sequence and determine phylogeny</h3>
	<img class="stretch" src="/images/talks/infection_tree_3b.png">
</section>

<section data-transition="none-in slide-out">
	<h3>Sequence and determine phylogeny</h3>
	<img class="stretch" src="/images/talks/infection_tree_4b.png">
</section>

<section style="text-align: left;">
	<h3>Pathogen genomes may reveal:</h3>
	<p>
	<ul>
		<li>Evolution of new adaptive variants</li>
		<li>Epidemic origins</li>
		<li>Patterns of geographic spread</li>
		<li>Animal-to-human spillover</li>
		<li>Transmission chains</li>
	</ul>
	</p>
</section>

<!--- Applications --->

<section>
	<h3><b>Influenza:</b> Forecasting spread of new variants for vaccine strain selection</h3>
	<img class="stretch" src="/images/talks/nextflu_apr2018_h3n2_tree_titers.png">
	<div class="citation">
		<a href="/papers/bedford-feb-2018-flu-report/">Bedford and Neher. 2018. bioRxiv.</a>
	</div>
</section>

<section>
	<h3><b>Zika:</b> Uncovering origins of the epidemic in the Americas</h3>
	<img class="stretch" src="/images/talks/zika_zibra_tree.png">
	<div class="citation">
		<a href="/papers/faria-zika-in-brazil/">Faria et al. 2017. Nature.</a>
	</div>
</section>

<section>
	<h3><b>Ebola:</b> Revealing spatial spread and persistence in West Africa</h3>
	<img class="stretch" src="/images/talks/ebola_mig_exploded_tree_wide.png">
	<div class="citation">
		<a href="/papers/dudas-ebola-epidemic-spread/">Dudas et al. 2017. Nature.</a>
	</div>
</section>

<section>
	<h3><b>MERS:</b> Quantifying camel-to-human spillover</h3>
	<img class="stretch" src="/images/talks/mers_tree.png">
	<div class="citation">
		<a href="/papers/dudas-mers-structure/">Dudas et al. 2018. eLife.</a>
	</div>
</section>

<section>
	<h3><b>TB:</b> Tracking individual transmission chains</h3>
	<img class="stretch" src="/images/talks/tb_transmission_tree.png">
	<div class="citation">
		<a href="https://doi.org/10.1093/molbev/msu121">Didelot et al. 2014. Mol Biol Evol.</a>
	</div>
</section>

<!--- Nextstrain --->

<section data-background="#000">
	<h1 class="title">Actionable inferences</h1>
</section>

<section>
	<h3>Genomic analyses are mostly done in a retrospective manner</h3>
	<img class="stretch" data-src="/images/talks/ebola_sampling_rugplots.png">
	<div class="citation">
		Dudas and Rambaut 2016
	</div>
</section>

<section>
	<h1>
		<span class="c1s">N</span><span class="c2s">e</span><span class="c3s">x</span><span class="c4s">t</span><span class="c5s">s</span><span class="c6s">t</span><span class="c7s">r</span><span class="c8s">a</span><span class="c9s">i</span><span class="c10s">n</span>
	</h1>
	<p>Project to conduct real-time molecular epidemiology and evolutionary analysis of emerging epidemics</p>
	<br>
	<p class="smaller"><i>
		with
		<img width=40 class="img-avatar" src="/images/team/richard-neher.jpg"> Richard Neher,
		<img width=40 class="img-avatar" src="/images/team/james-hadfield.jpg"> James Hadfield,
		<img width=40 class="img-avatar" src="/images/team/emma-hodcroft.jpg"> Emma Hodcroft,
		<img width=40 class="img-avatar" src="/images/team/thomas-sibley.jpg"> Tom Sibley,
		<br>
		<img width=40 class="img-avatar" src="/images/team/john-huddleston.jpg"> John Huddleston,
		<img width=40 class="img-avatar" src="/images/team/colin-megill.jpg"> Colin Megill,
		<img width=40 class="img-avatar" src="/images/team/sidney-bell.jpg"> Sidney Bell,
		<img width=40 class="img-avatar" src="/images/team/barney-potter.jpg"> Barney Potter,
		<br>
		<img width=40 class="img-avatar" src="/images/team/charlton-callender.jpg"> Charlton Callender
	</i></p>
</section>



<section data-background="#000">
	<h1 style="font-weight: 500;">
		<a href="http://nextstrain.org/" target="_blank">
			<span class="c1s">n</span><span class="c2s">e</span><span class="c3s">x</span><span class="c4s">t</span><span class="c5s">s</span><span class="c6s">t</span><span class="c7s">r</span><span class="c8s">a</span><span class="c9s">i</span><span class="c10s">n</span><span class="text-light-gray">.org</span>
		</a>
	</h1>
</section>


<section style="text-align: left;">
	<h3>Recently extended to include TB</h3>
	<p>
	<ul>
		<li>Now takes VCF input, allowing for a variety of bacteria to be included</li>
		<li>Resistance calls based on genotype</li>
		<li>Lack of recombination in TB makes it a good target for phylogenetic analysis, things like Klebsiella are much more challenging</li>
		<li>Examples for <a href="https://nextstrain.org/tb/global" target="_blank">global TB evolution</a> and TB in British Columbia, showing <a href="https://nextstrain.org/community/emmahodcroft/tb/GardyAll?c=Ethionamide&d=tree,entropy&p=full" target="_blank">preexisting transmitted resistance</a> as well as <a href="https://nextstrain.org/community/emmahodcroft/tb/GardyAll?c=Fluoroquinolones&d=tree,entropy&p=full" target="_blank">recent resistance clusters</a></li>
	</ul>
	</p>
</section>

<section style="text-align: left;">
	<h3>TB as Trojan Horse to cultivate genomic sequencing in public health</h3>
	<p>
	<ul>
		<li>Direct sequencing from clinical specimen avoids the multiple week delay of culture</li>
		<li>Clinically actionable result of treatment</li>
		<li>Databases of TB genomes to (1) provide critical training data for resistance calls and (2) provide epidemiological context for spread of TB and AMR</li>
	</ul>
</section>

<section>
	<img class="stretch" src="/images/talks/amr_cryptic.png">
</section>

<section>
	<h3>Genomic surveillance of AMR</h3>
	<p>Feedback as genomic data is used to improve clinical diagnostics and also improve epidemiological understanding</p>
</section>

<section>
	<h3>Global spread AMR</h3>
	<img class="stretch" src="/images/talks/amr_global.png">
	<div class="citation">
		<a href="https://science.sciencemag.org/content/360/6390/733.abstract">Baker et al. 2018. Science.</a>
	</div>
</section>

<section>
	<h3>Spread of MRSA</h3>
	<img class="stretch" src="/images/talks/amr_mrsa.png">
	<div class="citation">
		<a href="https://science.sciencemag.org/content/360/6390/733.abstract">Baker et al. 2018. Science.</a>
	</div>
</section>
