---
layout: talk_reveal3
title: Potential for antigenic drift in SARS-CoV-2
venue: Fred Hutch
event: HB Friday Faculty Talk
author: Trevor Bedford
widescreen: true
---

<!-- Outline:
-
-->

<!-- Total time of ~40 minutes + 20 minutes for questions -->

<section data-background-iframe="/talks/viz/gradient/">
	<h2 class="title">{{ page.title }}</h2>
	<br>&nbsp;<br>
	<p class="title">
	Trevor Bedford (@trvrb)
	<br>
	{{ page.date | date: "%-d %b %Y" }}
	<br>
	{{ page.event }}
	<br>
	{{ page.venue }}
	<br>&nbsp;<br>
	Slides at: bedford.io/talks
</section>

<section data-background="#0A0F12">
	<img class="stretch" src="/images/talks/shits_on_fire_yo.jpg">
</section>

<section>
	<h3>How likely is it that antigenic variants of SARS-CoV-2 will emerge, circulate and thereby decrease vaccine efficacy?</h3>
</section>

<!-- flu forecasting -->

<section>
	<h3>Integrating genotypes and phenotypes improves long-term forecasts of seasonal influenza A/H3N2 evolution</h3>
	<p class="smaller"><i>
		with <img width=40 class="img-avatar" src="/images/team/john-huddleston.jpg"> John Huddleston, Richard Neher, Dave Wentworth, Becky Kondor, John McCauley, Hideki Hasegawa, Kanta Subbarao and others
	</i></p>
	<div class="citation">
		<a href="/papers/huddleston-flu-forecasting/">Huddleston et al. 2020. eLife.</a>
	</div>
</section>

<section>
	<h3>Influenza viruses undergo antigenic drift</h3>
	<img class="stretch" data-src="/images/talks/bmds_lineages_drift.png">
	<div class="citation">
		<a href="/papers/bedford-flux/">Bedford et al. 2014. eLife.</a>
	</div>
</section>

<section>
	<h3>H3N2 vaccine updates occur every ~2 years</h3>
	<img class="stretch" data-src="/images/talks/flu_vaccine_updates_2020_12.png">
</section>

<section>
	<h3>Vaccine strain selection by WHO</h3>
	<img class="stretch" data-src="/images/talks/flu_vaccine_schedule.jpg">
</section>

<section>
	<h3>Select one strain from circulating diversity</h3>
	<img class="stretch" data-src="/images/talks/flu_h3n2_tree_vaccines_2008_2020.png">
</section>

<section>
	<h3>Fitness models project strain frequencies</h3>
	<p>Future frequency $x_i(t+\Delta t)$ of strain $i$ derives from strain fitness $f_i$ and present day frequency $x_i(t)$, such that</p>
	<p>$$\hat{x}_i(t+\Delta t) = x_i(t) \, \mathrm{exp}(f_i \, \Delta t)$$</p>
	<p>
		Total strain frequencies at each timepoint are normalized.
		This captures clonal interference between competing lineages.
	</p>
	<img class="stretch" data-src="/images/talks/clonal_interferance.jpg">
	<div class="citation">
		<a href="http://www.nature.com/nature/journal/v507/n7490/full/nature13087.html">&#321;uksza and L&auml;ssig. 2014. Nature.</a>
	</div>
</section>

<section>
	<h3>Match strain forecast to retrospective circulation</h3>
	<img class="stretch" data-src="/images/talks/prediction_fitness_model_schematic.png">
	<div class="citation">
		<a href="/papers/huddleston-flu-forecasting/">Huddleston et al. 2020. eLife.</a>
	</div>
</section>

<section>
	<h3>Train in 6-year sliding windows from 1995 to 2015 with most recent years held out as test</h3>
	<img class="stretch" data-src="/images/talks/prediction_cross_validation_strategy.png">
	<div class="citation">
		<a href="/papers/huddleston-flu-forecasting/">Huddleston et al. 2020. eLife.</a>
	</div>
</section>

<section>
	<h3>Composite models favor combinations of HI drift, local branching index and non-epitope fitness</h3>
	<img class="stretch" data-src="/images/talks/prediction_performance_composite.png">
	<div class="citation">
		<a href="/papers/huddleston-flu-forecasting/">Huddleston et al. 2020. eLife.</a>
	</div>
</section>

<section>
	<h3>Model successfully predicts clade growth and best pick from model is generally close to best possible retrospective pick</h3>
	<img class="stretch" data-src="/images/talks/prediction_validation_clade_growth_ranking.png">
	<div class="citation">
		<a href="/papers/huddleston-flu-forecasting/">Huddleston et al. 2020. eLife.</a>
	</div>
</section>

<section>
	<h3>Forecasts are live at <a href="https://nextstrain.org/flu">nextstrain.org/flu</a>, but are currently fraught due to lack of recent isolates</h3>
	<img class="stretch" data-src="/images/talks/nextflu_dec2020_h3n2_tree_clades.png">
</section>

<!-- seasonal CoV adaptive evolution -->

<section>
	<h3>Evidence for adaptive evolution in the receptor-binding domain of seasonal coronaviruses</h3>
	<p class="smaller"><i>
		with <img width=40 class="img-avatar" src="/images/team/katie-kistler.jpg"> Katie Kistler
	</i></p>
	<div class="citation">
		<a href="/papers/kistler-hcov-adaptive-evolution/">Kistler and Bedford. 2020. bioRxiv.</a>
	</div>
</section>

<section>
	<h3>Four species of seasonal coronavirus (OC43, 229E, NL63, HKU1) cause ~30% of common colds</h3>
	<img class="stretch" data-src="/images/talks/seattleflu_pathogen_prevalence_2020_12_10.png">
	<div class="citation">
		Seattle Flu Study. 2020.
	</div>
</section>

<!-- The S1 subunit contains a receptor-binding domain that recognizes and binds to the
host receptor angiotensin-converting enzyme 2 ACE-2, while the S2 subunit mediates viral
cell membrane fusion by forming a six-helical bundle via the two-heptad repeat domain. -->

<section>
	<h3>Focus on spike protein as target for immune response and antigenic escape</h3>
	<img class="stretch" data-src="/images/talks/sars_cov_2_spike_structure.png">
	<div class="citation">
		<a href="https://science.sciencemag.org/content/367/6483/1260">Wrapp et al. 2020. Science.</a>
	</div>
</section>

<section>
	<h3>Spike gene phylogenies for OC43 and 229E</h3>
	<img class="stretch" data-src="/images/talks/seasonal_cov_trees_oc43_229e.png">
	<div class="citation">
		<a href="/papers/kistler-hcov-adaptive-evolution/">Kistler and Bedford. 2020. bioRxiv.</a>
	</div>
</section>

<section>
	<h3>Substitutions occur more often in S1 subunit than S2</h3>
	<img class="stretch" data-src="/images/talks/seasonal_cov_mut_dist_oc43_229e.png">
	<div class="citation">
		<a href="/papers/kistler-hcov-adaptive-evolution/">Kistler and Bedford. 2020. bioRxiv.</a>
	</div>
</section>

<section>
	<h3>Nonsynonymous substitutions accumulate rapidly in exposed portions of spike</h3>
	<img class="stretch" data-src="/images/talks/seasonal_cov_divergence_oc43_229e.png">
	<div class="citation">
		<a href="/papers/kistler-hcov-adaptive-evolution/">Kistler and Bedford. 2020. bioRxiv.</a>
	</div>
</section>

<section>
	<h3>Diversity and divergence used to estimate accumulation of adaptive substitutions</h3>
	<img class="stretch" data-src="/images/talks/seasonal_cov_adaptation_accumulation_oc43.png">
	<div class="citation">
		<a href="/papers/kistler-hcov-adaptive-evolution/">Kistler and Bedford. 2020. bioRxiv.</a>
	</div>
</section>

<section>
	<h3>This analysis can be summarized as rate of adaptive substitutions per year (about ~0.5 adaptive subs per year for OC43)</h3>
	<img class="stretch" data-src="/images/talks/seasonal_cov_adaptation_rate_oc43_229e.png">
	<div class="citation">
		<a href="/papers/kistler-hcov-adaptive-evolution/">Kistler and Bedford. 2020. bioRxiv.</a>
	</div>
</section>

<section>
	<h3>And compared to rates of adaptive substitutions in other viruses</h3>
	<img class="stretch" data-src="/images/talks/seasonal_cov_adaptation_rate_comparison.png">
	<div class="citation">
		<a href="/papers/kistler-hcov-adaptive-evolution/">Kistler and Bedford. 2020. bioRxiv.</a>
	</div>
</section>

<section>
	<h3>
		Adaptive evolution in spike is clearly present in OC43 and 229E.
		It's likely this results in antigenic evolution and contributes to reinfection
		observed to occur every ~3 years.
	</h3>
</section>

<!-- SARS-CoV-2 evolution -->

<section data-background="#000000">
	<h1 class="title">SARS-CoV-2</h1>
</section>

<section>
	<h3>Sequencing and data sharing in almost real-time, now with over 250k genomes</h3>
	<img class="stretch" src="/images/talks/ncov_gisaid_2020_07_15_cadence.png">
	<div class="citation">
		Figure by Hadfield and Hodcroft using data from GISAID
	</div>
</section>

<section>
	<h3>SARS-CoV-2 lineages establish globally in February and March</h3>
	<img class="stretch" src="/images/talks/ncov_tree_map_early_pandemic.png">
	<div class="citation">
		<a href="/papers/bedford-ncov-cryptic-transmission/">Bedford et al. 2020. Science.</a>
	</div>
</section>

<section>
	<div class="inset-top-right">
		With ongoing mitigation and <br> decreased international travel, <br> regional clades have emerged
	</div>
	<img class="stretch" src="/images/talks/ncov_tree_2020_10_20_global_jun_oct.png">
	<div class="citation">
		<a href="https://nextstrain.org">nextstrain.org</a>
	</div>
</section>

<section>
	<div class="inset-top-right">
		With ongoing mitigation and <br> decreased international travel, <br> regional clades have emerged
	</div>
	<img class="stretch" src="/images/talks/ncov_tree_2020_10_20_asia_jun_oct.png">
	<div class="citation">
		<a href="https://nextstrain.org">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>Limited early mutations like D614G spread globally during initial wave</h3>
	<img class="stretch" src="/images/talks/ncov_d614g_european_introduction.png">
	<div class="citation">
		<a href="https://nextstrain.org">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>More recent variants are confined to regional dominance</h3>
	<img class="stretch" data-src="/images/talks/ncov_a222v_tree_map.jpg">
	<div class="citation">
		<a href="https://www.medrxiv.org/content/10.1101/2020.10.25.20219063v1">Hodcroft et al. 2020. medRxiv.</a>
	</div>
</section>

<!--
Variable positions in spike:
https://nextstrain.org/ncov/global?c=gt-S_5,12,18,69,80,98,138,153,215,222,439,477,484,501,614,681,701

Variable positions in the RBD, which is 318 to 510:
https://nextstrain.org/ncov/global?c=gt-S_439,477,484,501
-->

<section>
	<h3>Starting to see circulating spike variants</h3>
	<img class="stretch" data-src="/images/talks/ncov_spike_variants_global_2020_12_10.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov/global?c=gt-S_5,12,18,69,80,98,138,153,215,222,439,477,484,501,614,681,701">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>A222V, N439K and S477N spreading in Europe (although may be largely explained by epidemiology)</h3>
	<img class="stretch" data-src="/images/talks/ncov_spike_variants_europe_2020_12_10.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov/europe?c=gt-S_222,439,477">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>S477N also spreading in Oceania, having arisen independently</h3>
	<img class="stretch" data-src="/images/talks/ncov_spike_variants_oceania_2020_12_10.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov/oceania?c=gt-S_222,477">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>N439K shows evidence of reduced binding and neutralization</h3>
	<img class="stretch" data-src="/images/talks/sars_cov_2_n439k_thomson.png">
	<div class="citation">
		<a href="https://www.biorxiv.org/content/10.1101/2020.11.04.355842v1">Thomson et al. 2020. medRxiv.</a>
	</div>
</section>

<section>
	<h3>Y453F shows associated with passage in farmed mink</h3>
	<img class="stretch" data-src="/images/talks/ncov_tree_mink_2020_12_10.png">
	<div class="citation">
		<a href="https://nextstrain.org/groups/neherlab/ncov/S.Y453F?c=gt-S_453">Nextstrain analysis by Emma Hodcroft</a>
	</div>
</section>

<section data-background="#000000">
	<h3 class="title">
		I think SARS-CoV-2 will undergo antigenic drift
	</h3>
</section>

<section>
	<h3>Strong immune response to mRNA vaccines may help here</h3>
	<img class="stretch" data-src="/images/talks/sars_cov_2_bnt162b_titers.jpg">
	<div class="citation">
		<a href="https://www.nejm.org/doi/full/10.1056/NEJMoa2027906">Walsh et al. 2020. NEJM.</a>
	</div>
</section>

<section style="text-align: left; margin-left: 20px;">
	<h3>Monitoring for antigenic drift</h3>
	<p>
	<ul>
		<li>Test neutralization of circulating variants against serum panels from <br> recovered and vaccinated individuals</li>
		<li>Combine vaccination records with sequencing of clinical samples</li>
	</ul>
	</p>
</section>

<!------------------------------->

<section>
	<h3>Acknowledgements</h3>

	<p class="tiny">
		<b>Seasonal flu</b>: WHO Global Influenza Surveillance Network, GISAID,
		John Huddleston, Richard Neher, Jover Lee, Dave Wentworth, Becky Kondor
	</p>
	<p class="tiny">
		<b>Seasonal CoVs</b>: Katie Kistler
	</p>
	<p class="tiny">
		<b>SARS-CoV-2 genomic epi</b>: Data producers from all over the world, GISAID and the Nextstrain team
	</p>
	<p class="tiny"><b>Bedford Lab</b>:
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/allison-black.jpg"> Alli Black,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/john-huddleston.jpg"> John Huddleston,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/james-hadfield.jpg"> James Hadfield,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/katie-kistler.jpg"> Katie Kistler,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/louise-moncla.jpg"> Louise Moncla,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/maya-lewinsohn.jpg"> Maya Lewinsohn,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/thomas-sibley.jpg"> Thomas Sibley,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/jover-lee.jpg"> Jover Lee,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/kairsten-fay.jpg"> Kairsten Fay,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/misja-ilcisin.jpg"> Misja Ilcisin,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/cassia-wagner.jpg"> Cassia Wagner,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/miguel-paredes.jpg"> Miguel Paredes,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/nicola-mueller.jpg"> Nicola Müller,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/marlin-figgins.jpg"> Marlin Figgins,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/eli-harkins.jpg"> Eli Harkins</span>
	</p>

	<div class="pull-right" style="vertical-align: bottom; display: table-cell;">
		<img width="11%" data-src="/images/talks/logo_fred_hutch_vertical.png">
		&nbsp; &nbsp;
		<img width="8%" data-src="/images/talks/logo_pew.png">
		&nbsp; &nbsp;
		<img width="14%" data-src="/images/talks/logo_bmgf.png">
		&nbsp; &nbsp;
		<img width="12%" data-src="/images/talks/logo_gates_ventures.png">
		&nbsp; &nbsp;
		<img width="11%" data-src="/images/talks/logo_wellcome_trust.jpg">
		&nbsp; &nbsp;
		<img width="7%" data-src="/images/talks/logo_nih.jpg">
		&nbsp; &nbsp;
		<img width="12%" data-src="/images/talks/logo_osp.jpg">
	</div>
</section>
