---
layout: talk_reveal3
title: Tracking novel coronavirus epidemic spread through genome sequencing
venue: Seattle, WA
event: AAAS Annual Meeting
author: Trevor Bedford
---

<!-- Total time of 20 minutes. -->

<section data-background-iframe="/talks/viz/gradient/">
	<h2 class="title">{{ page.title }}</h2>
	<br>&nbsp;<br>
	<p class="title">
	Trevor Bedford (@trvrb)
	<br>
	{{ page.date | date: "%-d %b %Y" }}
	<br>
	{{ page.event }}
	<br>
	{{ page.venue }}
	<br>&nbsp;<br>
	Slides at: bedford.io/talks
</section>

<section>
	<h3>Traditionally, we learn about an epidemic by tracking cases</h3>
	<img class="stretch" data-src="/images/talks/ncov_epi_curve_2020_02_13.png">
	<div class="citation">
		China National Health Commission
	</div>
</section>

<section>
	<h2>Genetic sequencing to reconstruct pathogen spread</h2>
</section>

<section data-transition="slide-in none-out">
	<h3>Epidemic process</h3>
	<img class="stretch" src="/images/talks/infection_tree_1.png">
</section>

<section data-transition="none">
	<h3>Sample some individuals</h3>
	<img class="stretch" src="/images/talks/infection_tree_2.png">
</section>

<section data-transition="none">
	<h3>Sequence and determine phylogeny</h3>
	<img class="stretch" src="/images/talks/infection_tree_3b.png">
</section>

<section data-transition="none-in slide-out">
	<h3>Sequence and determine phylogeny</h3>
	<img class="stretch" src="/images/talks/infection_tree_4b.png">
</section>

<section>
	<h3>Revolutionary tool for understanding outbreaks</h3>
	<img class="stretch" src="/images/talks/genomic_epi_zika_ebola.png">
	<div class="citation">
		<a href="/papers/faria-zika-in-brazil/">Faria et al. 2017. Nature.</a>
		<a href="/papers/dudas-ebola-epidemic-spread/">Dudas et al. 2017. Nature.</a>
	</div>
</section>

<section>
	<h3>Genomic analyses were mostly done in a retrospective manner</h3>
	<img class="stretch" data-src="/images/talks/ebola_sampling_rugplots.png">
	<div class="citation">
		Dudas and Rambaut 2016
	</div>
</section>

<section style="text-align: left;">
	<h3>Key challenges to making genomic epidemiology actionable</h3>
	<p>
	<ul>
		<li>Timely analysis and sharing of results critical</li>
		<li>Dissemination must be scalable</li>
		<li>Integrate many data sources</li>
		<li>Results must be easily interpretable and queryable</li>
	</ul>
	</p>
</section>

<section>
	<h1>
		<span class="c1s">N</span><span class="c2s">e</span><span class="c3s">x</span><span class="c4s">t</span><span class="c5s">s</span><span class="c6s">t</span><span class="c7s">r</span><span class="c8s">a</span><span class="c9s">i</span><span class="c10s">n</span>
	</h1>
	<p>Project to conduct real-time molecular epidemiology and evolutionary analysis of emerging epidemics</p>
	<br>
	<p class="smaller"><i>
		with
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/richard-neher.jpg"> Richard Neher,</span>
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/james-hadfield.jpg"> James Hadfield,</span>
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/emma-hodcroft.jpg"> Emma Hodcroft,</span>
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/thomas-sibley.jpg"> Thomas Sibley,</span>
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/john-huddleston.jpg"> John Huddleston,</span>
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/louise-moncla.jpg"> Louise Moncla,</span>
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/misja-ilcisin.jpg"> Misja Ilcisin,</span>
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/kairsten-fay.jpg"> Kairsten Fay,</span>
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/jover-lee.jpg"> Jover Lee,</span>
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/allison-black.jpg"> Allison Black,</span>
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/colin-megill.jpg"> Colin Megill,</span>
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/sidney-bell.jpg"> Sidney Bell,</span>
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/barney-potter.jpg"> Barney Potter,</span>
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/charlton-callender.jpg"> Charlton Callender</span>
	</i></p>
</section>

<!-- <section>
	<h3>Nextstrain architecture</h3>
	<p>All code open source at <a class="off" href="https://github.com/nextstrain">github.com/nextstrain</a></p>
	<p>Two central aims: (1) <i>rapid and flexible phylodynamic analysis</i> and <br> (2) <i>interactive visualization</i></p>
	<img class="stretch" src="/images/talks/nextstrain_components.png">
</section>

<section style="text-align: left;">
	<h3>Nextstrain is two things</h3>
	<p>
	<ul>
		<li>a bioinformatics toolkit and visualization app, which can be used for a broad range of datasets</li>
		<li>a collection of real-time pathogen analyses kept up-to-date on the website nextstrain.org</i></li>
	</ul>
	</p>
</section>

<section data-background="#000">
	<h1 style="font-weight: 500;">
		<a href="http://nextstrain.org/" target="_blank">
			<span class="c1s">n</span><span class="c2s">e</span><span class="c3s">x</span><span class="c4s">t</span><span class="c5s">s</span><span class="c6s">t</span><span class="c7s">r</span><span class="c8s">a</span><span class="c9s">i</span><span class="c10s">n</span><span class="text-light-gray">.org</span>
		</a>
	</h1>
</section> -->

<!-- nCoV -->

<section>
	<h3>Jan 10: nCoV is a SARS-like betacoronavirus</h3>
	<img class="stretch" data-src="/images/talks/ncov_beta_cov_tree.png">
	<div class="citation">
		<a href="https://nextstrain.org/groups/blab/beta-cov">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>Jan 11: These viruses have a natural reservoir in bats</h3>
	<img class="stretch" data-src="/images/talks/ncov_sars_like_tree_host.png">
	<div class="citation">
		<a href="https://nextstrain.org/groups/blab/sars-like-cov">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>Jan 11: Initial 5 nCoV genomes from Wuhan showed highly restricted genetic diversity</h3>
	<img class="stretch" data-src="/images/talks/ncov_tree_2020_01_11.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>
		Initially thought clustering due to epi investigation of linked cases at
		Huanan seafood market
	</h3>
</section>

<section>
	<h3>Jan 17: Additional 2 nCoV genomes from Thailand travel cases also lacked diversity</h3>
	<img class="stretch" data-src="/images/talks/ncov_tree_2020_01_17.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>Jan 19: Additional 5 nCoV genomes from Wuhan still lacked diversity</h3>
	<img class="stretch" data-src="/images/talks/ncov_tree_2020_01_19.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov">nextstrain.org</a>
	</div>
</section>

<section>
	<h1>üòß</h1>
</section>

<section>
	<h3>
		Single introduction into the human population between Nov 15 and Dec 15 and
		human-to-human epidemic spread from this point forward
	</h3>
</section>

<section>
	<h3>
		Spent the week of Jan 20 alerting public health officials, and since then have
		aimed to keep nextstrain.org/ncov updated within ~1hr of new sequences being
		deposited
	</h3>
</section>

<section style="text-align: left;">
	<h3>Feb 13: >100 sequences shared by groups all over the world</h3>
	<img class="stretch" data-src="/images/talks/ncov_gisaid_map_2020_02_13.png">
</section>

<section>
	<h3>Almost real-time with many genomes shared within 3-6 days of sampling</h3>
	<img class="stretch" data-src="/images/talks/ncov_gisaid_2020_02_04.png">
	<div class="citation">
		<a href="https://gisaid.org">gisaid.org</a>
	</div>
</section>

<section>
	<h3>
		Current state of
		<a href="https://nextstrain.org/ncov" target="_blank">nextstrain.org/ncov</a>
		with 88 genomes as of Feb 13
	</h3>
</section>

<section>
	<h2>Phylodynamic estimates of epidemic growth</h2>
</section>

<section>
	<h3>Rate of evolution and time of common ancestor</h3>
	<img class="stretch" data-src="/images/talks/ncov_phylodynamics_clock_tmrca_2020_02_09.png">
	<div class="citation">Virological.org</div>
</section>

<section>
	<h3>Viral genetic diversity and population size</h3>
	<img class="stretch" data-src="/images/talks/ncov_phylodynamics_netau_2020_02_09.png">
	<div class="citation">Virological.org</div>
</section>

<section>
	<h3>Transformed to infection prevalence</h3>
	<img class="stretch" data-src="/images/talks/ncov_phylodynamics_prevalence_2020_02_09.png">
	<div class="citation">Virological.org</div>
</section>

<section>
	<h3>And to cumulative incidence</h3>
	<img class="stretch" data-src="/images/talks/ncov_phylodynamics_incidence_2020_02_09.png">
	<div class="citation">Virological.org</div>
</section>

<section>
	<h3>
		Scientific communication surrounding outbreak has completely flipped with
		everything posted to bioRxiv, modeling groups posting live analyses and
		crowd-sourced line lists üôåüèª
	</h3>
</section>

<section>
	<h3>
		This communication between academics and public health officials has spilled
		over with huge interest from general public
	</h3>
</section>

<section>
	<h3>
		This is having knock-on effects on science communication and spread of
		misinformation
	</h3>
	<img class="stretch" data-src="/images/talks/ncov_hiv_bunk.png">
</section>

<section>
	<img class="stretch" data-src="/images/talks/ncov_hiv_debunk.png">
	<div class="citation">
		<a href="https://twitter.com/trvrb/status/1223666856923291648">Twitter</a>
	</div>
</section>

<section style="text-align: left;">
	<h3>Moving forward</h3>
	<p>
	<ul>
		<li>Epidemic spread still appears to be sustained with estimates of R0 between ~2-3 and epidemic
			doubling time of ~6 days</li>
		<li>We have not yet had time to ascertain effects of intervention measures, but my hope for
			containment is slim</li>
		<li>Biggest question for me now surrounds severity</li>
		<li>I expect genomic data to be most immediately useful to help pin down emerging community
			transmission</li>
	</ul>
	</p>
</section>

<section>
	<h3>Acknowledgements</h3>

	<p class="even-tinier">
		<b>Data producers from all over the world</b>: BGI & Institute of Microbiology, Chinese Academy of Sciences & Shandong First Medical University & Shandong Academy of Medical Sciences; Beijing Genomics Institute (BGI), Shenzhen, China; Centers for Disease Control, R.O.C., Taipei, Taiwan; Charite Universitaetsmedizin Berlin, Berlin, Germany; Chongqing Municipal Center for Disease Control and Prevention, Chongqing, China; Victorian Infectious Disease Reference Laboratory, Melbourne, Australia; National Institute of Health, Nonthaburi, Thailand; Guangdong Provincial Center for Diseases Control and Prevention; Guangdong Provinical Public Health, Guangzhou, China; Department of Microbiology, Zhejiang Provincial Center for Disease Control and Prevention, Hangzhou, China; Department of Virology, University of Helsinki, Helsinki, Finland; Guangdong Provincial Center for Diseases Control and Prevention, Guangzhou, China; Hangzhou Center for Disease and Control Microbiology Lab, Zhejiang, China; Hubei Provincial Center for Disease Control and Prevention, Wuhan, China; Chinese Academy of Medical Sciences & Peking Union Medical College, Beijing, China; Institute of Viral Disease Control and Prevention, China CDC, Beijing, China; KU Leuven, Leuven, Belgium; Korea Centers for Disease Control & Prevention (KCDC), Cheongju, Korea; Microbial Genomics Core Lab, National Taiwan University, Taipei, Taiwan; NSW Health Pathology - Institute of Clinical Pathology and Medical Research; Centre for Infectious Diseases and Microbiology Laboratory Services; Westmead Hospital; University of Sydney, Westmead, Australia; University of Sydney, Westmead, Australia; National Institute of Hygiene and Epidemiology, Hanoi, Vietnam; National Institute for Communicable Disease Control and Prevention (ICDC), China CDC, Beijing, China; National Institute for Viral Disease Control and Prevention, China CDC, Beijing, China; National Public Health Laboratory, Singapore; Institut Pasteur, Paris, France; Centers for Disease Control and Prevention, Atlanta, USA; National Institute of Infectious Diseases, Tokyo, Japan; Duke-NUS Medical School, Singapore; Public Health Virology Laboratory, Brisbane, Australia; Public Health England, London, United Kingdom; National Clinical Research Center for Infectious Disease, Shenzhen, China; Wuhan Institute of Virology, Chinese Academy of Sciences, Wuhan, China, made possible through GISAID
	</p>

	<p class="even-tinier">
		<b>Nextstrain</b>: Richard Neher, James Hadfield, Emma Hodcroft, Tom Sibley,
		John Huddleston, Louise Moncla, Misja Ilcisin, Kairsten Fay, Jover Lee, Allison Black
		Colin Megill, Sidney Bell, Barney Potter, Charlton Callender
	</p>

	<div class="pull-right" style="vertical-align: bottom; display: table-cell;">
		<img width="14%" data-src="/images/talks/logo_fred_hutch_vertical.png">
		&nbsp; &nbsp;
		<img width="10%" data-src="/images/talks/logo_pew.png">
		&nbsp; &nbsp;
		<img width="16%" data-src="/images/talks/logo_bmgf.png">
		&nbsp; &nbsp;
		<img width="13%" data-src="/images/talks/logo_wellcome_trust.jpg">
		&nbsp; &nbsp;
		<img width="9%" data-src="/images/talks/logo_nih.jpg">
		&nbsp; &nbsp;
		<img width="14%" data-src="/images/talks/logo_osp.jpg">
	</div>
</section>
