---
layout: talk_reveal3
title: SARS-CoV-2 genomic nomenclature
venue: WHO
event: SARS-CoV-2 Evolution Working Group
author: Trevor Bedford
widescreen: true
---

<section data-background-iframe="/talks/viz/gradient/">
	<h2 class="title">{{ page.title }}</h2>
	<br>&nbsp;<br>
	<p class="title">
	Trevor Bedford (@trvrb)
	<br>
	Associate Professor, Fred Hutchinson Cancer Research Center
	<br>
	{{ page.date | date: "%-d %b %Y" }}
	<br>
	{{ page.event }}
	<br>
	{{ page.venue }}
	<br>&nbsp;<br>
	Slides at: bedford.io/talks
</section>

<section>
	<h3>Overview of clade/lineage assignments</h3>
	<img class="stretch" data-src="/images/talks/ncov_nomenclature_overview_trees.png">
</section>

<section>
	<h3>Distribution of clade/lineage assignments</h3>
	<p>
	<ul>
		<li>5 Nextstrain clades (20A, 20B, 20C, 20G, 20I) comprise ~80% of circulating viruses</li>
		<li>2 GISAID clades (GH, GR) comprise ~80% of circulating viruses</li>
		<li>5 PANGO lineages (B.1, B.1.1.7, B.1.2, B.1.177, B.1.351) comprise ~50% of circulating viruses</li>
	</ul>
	</p>
	<img class="stretch" data-src="/images/talks/ncov_nomenclature_overview_frequencies.png">
</section>

<section>
	<h3>Nextstrain clade nomenclature</h3>
	<p>
	<ul>
		<li>Naming proceeds in hurricane-style <a href="https://nextstrain.org/blog/2021-01-06-updated-SARS-CoV-2-clade-naming">year-letter format</a>, ie 21A, 21B, 21C, etc...</li>
		<li>A clade is labeled when it reaches >20% global or >30% regional frequency for >2 months</li>
		<li>A named clade must be at least 2 nucleotide mutations distant from its parent clade</li>
		<li>A named clade is immediately labeled when a "variant of concern" (VOC) is recognized; these clades are dual-labeled with clade and variant name, eg 20H/501Y.V2</li>
	</ul>
	</p>
	<img class="stretch" data-src="/images/talks/ncov_nomenclature_clades_schematic.png">
</section>

<section>
	<h3>Label subclades by mutations</h3>
	<p>
	<ul>
		<li>The 12 named clades are necessarily coarse; subclades can be specified by including a "/ mutation" demarcation, eg 20G/8083A indicates viruses <a href="https://nextstrain.org/ncov/north-america?f_clade_membership=20G&gt=nuc.8083A">belonging to clade 20G that possess the genotype nuc 8083A</a></li>
		<li>Likewise subclade label 20C/S:452R indicates viruses <a href="https://nextstrain.org/ncov/north-america?f_clade_membership=20C&gt=S.452R">belonging to clade 20C that possess the genotype spike 452R</a></li>
		<li>Separately, I'd propose that this convention be generally adopted, eg B.1.1.7/S:484K</li>
		<li>This convention allows outside groups to point at specific clades without needing a centralized updating mechanism to flag every new clade of interest</li>
	</ul>
	</p>
</section>

<section>
	<h3>"California variant" as case study</h3>
	<p>
	<ul>
		<li>Groups notice a clade bearing spike mutations S13I, W152C and L452R that has been rising in frequency in Southern California and want to discuss this publicly and discuss it as "CAL.20C"</li>
		<li>At time of recognition, <a href="https://nextstrain.org/ncov/north-america/2021-01-15?branchLabel=aa&c=pangolin_lineage&gt=S.452R,152C">this clade is scattered across multiple PANGO lineages</a> and so there's no way for this "variant" to be announced with a matching PANGO designation</li>
		<li>Subsequently, PANGO nomenclature has been updated and this clade is now distributed between <a href="https://nextstrain.org/ncov/north-america/2021-01-29?branchLabel=aa&c=pangolin_lineage&gt=S.452R,152C">sister lineages B.1.427 and B.1.429</a>. Currently to the refer to the "California variant" in PANGO nomenclature, you'd say "a variant comprised of lineages B.1.427 and B.1.429".</li>
		<li>In Nextstrain nomenclature, this variant can be labeled at time of initial recognition as 20C/S:452R without any registration or updating required on our part</li>
	</ul>
	</p>
</section>

<section>
	<h3>Tension between few labels for vaccine strain selection and many labels for genomic epidemiology, but a middle ground is possible</h3>
	<p>
	<ul>
		<li>Clades/lineages should be monophyletic and distinct from one another to encourage accurate recall when assigning new genomes</li>
		<li>Demarcations should be coarse enough to faciliate accurate recall and parsable dynamics; I'd like fewer than 50 demarcations per year</li>
		<li>I have no strong preference between hurricane-style 21A names and hierarchical B.1.427 names</li>
		<li>Lean on "/ mutation" labeling to still allow specificity without having to have an overlong catalog of clades/lineages</li>
	</ul>
	</p>
</section>
