---
layout: talk_reveal4
title: SARS-CoV-2 evolutionary dynamics
venue: Massachusetts Consortium on Pathogen Readiness
event: Emerging Variants Meeting
author: Trevor Bedford
widescreen: true
---

<!-- ~45 min worth of slides -->

<!--
Outline:

- Nextstrain and real-time tracking of SARS-CoV-2 evolution
- Emergence and spread of VOCs
- Current circulation patterns
- Katie's adaptive evolution work
- Frequency dynamics / Marlin's work
- Perspectives on genomic surveillance
- Big picture (pessimistic) perspective for the future
-->

<!-- Color ramp: [#5097BA", "#60AA9E", "#75B681", "#8EBC66", "#AABD52", "#C4B945", "#D9AD3D", "#E59637", "#E67030", "#DF4327"] -->

<section data-transition="fade" data-background="#000">
	<h2 class="title">{{ page.title }}</h2>
	<br>&nbsp;<br>
	<p class="title">
	Trevor Bedford (@trvrb)
	<br>
	Associate Professor, Fred Hutchinson Cancer Research Center
	<br>
	{{ page.date | date: "%-d %b %Y" }}
	<br>
	{{ page.event }}
	<br>
	{{ page.venue }}
	<br>&nbsp;<br>
	Slides at: bedford.io/talks
</section>

<!-- Introduction -->

<section data-transition-speed="slow" data-background="/images/talks/ncov_tree_unrooted_2021_09_11.png">
</section>

<section class="left-align">
	<h3>1. Current circulation patterns</h3>
	<h3>2. Assessing adaptive evolution</h3>
	<h3>3. Variant transmission dynamics</h3>
	<h3>4. Potential for epistasis</h3>
	<h3>5. Future perspectives</h3>
</section>

<!-- Current circulation patterns -->

<section data-transition="fade" data-background="#5097BA">
	<h1 class="title">Current circulation patterns</h1>
</section>

<section>
	<h3>Spread of VOC / VOI lineages across the world</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_animation_2021_09_12.gif">
	<div class="citation">
		<a href="https://nextstrain.org/ncov/gisaid/global?animate=2019-12-19,2021-09-11,1,0,30000&branchLabel=none&d=tree,map&p=grid">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>Delta is outcompeting other variants and is on track to sweep</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_emerging_lineages_2021_09_12.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov/gisaid/global?c=emerging_lineage">nextstrain.org</a>
	</div>
</section>

<section>
	<div class="left" style="float:left; width:30%;">
		<h3>Locally emerging sublineages such as AY.4.2 with spike mutations 145H and 222V</h3>
	</div>
	<div class="right" style="float:right; width:69%;">
		<img data-src="/images/talks/ncov_variants_145H_uk_2021_10_19.png">
	</div>
	<div class="citation">
		<a href="https://nextstrain.org/groups/neherlab/ncov/united-kingdom?c=gt-S_145,222&f_country=United%20Kingdom&f_division=England,Scotland,Wales,Northern%20Ireland&p=grid&r=division">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>Variants show excess mutations across the genome</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_scatter_divergence_2021_09_12.png">
	<div class="citation">
		<a href="https://nextstrain.org/groups/blab/ncov/adaptive-evolution/2021-08-15?branches=hide&l=scatter&p=full&scatterY=div">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>But show most substantial excess in the S1 domain of spike</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_scatter_s1_mutations_2021_09_12.png">
	<div class="citation">
		<a href="https://nextstrain.org/groups/blab/ncov/adaptive-evolution/2021-08-15?branches=hide&l=scatter&p=full&scatterY=S1_mutations">nextstrain.org</a>
	</div>
</section>

<!-- Assessing adaptive evolution -->

<section data-transition="fade" data-background="#60AA9E">
	<h1 class="title">Assessing adaptive evolution</h1>
</section>

<section class="left-align">
	<h3>Rapid and parallel adaptive mutations in spike S1 drive clade success in SARS-CoV-2</h3>
	<div class="smaller"><i>
		<img width=50 class="img-avatar" src="/images/team/katie-kistler.jpg"> Katie Kistler, &hairsp;
		<img width=50 class="img-avatar" src="/images/team/john-huddleston.jpg"> John Huddleston
	</i></div>
	<p>&nbsp;<p>
</section>

<section>
	<h3>Phylogeny of 10k genomes equitably sampled in space and time</h3>
	<img class="stretch" data-src="/images/talks/sarscov2_adaptive_evolution_phylogeny.png">
	<div class="citation">
		<a href="/papers/kistler-sarscov2-adaptive-evolution/">Kistler et al. 2021. bioRxiv.</a>
	</div>
</section>

<section>
	<h3>Measure clade growth as a proxy for viral fitness</h3>
	<img class="stretch" data-src="/images/talks/sarscov2_adaptive_evolution_phylogeny_logistic_growth.png">
	<div class="citation">
		<a href="/papers/kistler-sarscov2-adaptive-evolution/">Kistler et al. 2021. bioRxiv.</a>
	</div>
</section>

<section>
	<h3>Clades with more S1 nonsynonymous mutations grow faster</h3>
	<img class="stretch" data-src="/images/talks/sarscov2_adaptive_evolution_mut_accumulation_s1.png">
	<div class="citation">
		<a href="/papers/kistler-sarscov2-adaptive-evolution/">Kistler et al. 2021. bioRxiv.</a>
	</div>
</section>

<section>
	<h3>This correlation is absent in control regions</h3>
	<img class="stretch" data-src="/images/talks/sarscov2_adaptive_evolution_mut_accumulation_control.png">
	<div class="citation">
		<a href="/papers/kistler-sarscov2-adaptive-evolution/">Kistler et al. 2021. bioRxiv.</a>
	</div>
</section>

<section>
	<h3>This correlation is stronger in S1 than other genes</h3>
	<img class="stretch" data-src="/images/talks/sarscov2_adaptive_evolution_mut_accumulation_genes.png">
	<div class="citation">
		<a href="/papers/kistler-sarscov2-adaptive-evolution/">Kistler et al. 2021. bioRxiv.</a>
	</div>
</section>

<section>
	<h3>S1 is quickly evolving and highly correlated with clade growth</h3>
	<img class="stretch" data-src="/images/talks/sarscov2_adaptive_evolution_mut_accumulation_all_genes.png">
	<div class="citation">
		<a href="/papers/kistler-sarscov2-adaptive-evolution/">Kistler et al. 2021. bioRxiv.</a>
	</div>
</section>

<section data-transition="slide-in none-out">
	<h3>dN/dS through time further highlights adaptive evolution</h3>
	<img class="stretch" data-src="/images/talks/sarscov2_adaptive_evolution_dnds.png">
	<div class="citation">
		<a href="/papers/kistler-sarscov2-adaptive-evolution/">Kistler et al. 2021. bioRxiv.</a>
	</div>
</section>

<section data-transition="none-in slide-out">
	<h3>Rapid pace of adaptive evolution relative to H3N2 influenza</h3>
	<img class="stretch" data-src="/images/talks/sarscov2_adaptive_evolution_dnds_h3n2.png">
	<div class="citation">
		<a href="/papers/kistler-sarscov2-adaptive-evolution/">Kistler et al. 2021. bioRxiv.</a>
	</div>
</section>

<section>
	<h3>S1 mutations cluster phylogenetically, suggesting bursts of evolution</h3>
	<img class="stretch" data-src="/images/talks/sarscov2_adaptive_evolution_wait_times.png">
	<div class="citation">
		<a href="/papers/kistler-sarscov2-adaptive-evolution/">Kistler et al. 2021. bioRxiv.</a>
	</div>
</section>

<section>
	<h3>Many S1 mutations show strong convergent evolution</h3>
	<img class="stretch" data-src="/images/talks/sarscov2_adaptive_evolution_convergent.png">
	<div class="citation">
		<a href="/papers/kistler-sarscov2-adaptive-evolution/">Kistler et al. 2021. bioRxiv.</a>
	</div>
</section>

<section class="left-align">
	<h3>
		Mutations in S1 arising via within-host pressures result increase viral fitness and are enriched in the viral population by natural selection
	</h3>
	<br>
	<h3>
		Although selection has not been primarily for antigenic drift, observed level of adaptability suggests its potential
	</h3>
</section>

<!-- Variant transmission dynamics -->

<section data-transition="fade" data-background="#75B681">
	<h1 class="title">Variant transmission dynamics</h1>
</section>

<section class="left-align">
	<h3>
		Multiple approaches to modeling fitness differences between circulating variants
	</h3>
</section>

<section>
	<h3>Multinomial logistic regression models work well on frequencies</h3>
	<img class="stretch" data-src="/images/talks/sarscov2_variant_rt_logistic_regression_2021_09_01.png">
</section>

<section>
	<h3>However, frequency of a variant may rise while cases fall</h3>
	<img class="stretch" data-src="/images/talks/sarscov2_variant_rt_partitioning_cases_2021_09_01.png">
</section>

<section class="left-align">
	<h3>SARS-CoV-2 variant dynamics across US states show consistent differences in transmission rates</h3>
	<div class="smaller"><i>
		<img width=50 class="img-avatar" src="/images/team/marlin-figgins.jpg"> Marlin Figgins
	</i></div>
	<p>&nbsp;<p>
</section>

<section class="left-align">
	<div class="left" style="float:left; width:35%">
		<h3>Estimation of variant-specific Rt through time using state-level data</h3>
		<p>State-level case counts are partitioned based on frequencies of sequenced cases</p>
	</div>
	<div class="right" style="float:right; width:60%">
		<img data-src="/images/talks/sarscov2_variant_rt_method_wa_2021_09_01.png">
	</div>
	<div class="citation">
		Figgins and Bedford. Unpublished.
	</div>
</section>

<section>
	<h3>Differences in intrinsic Rt across variants, but all trending downwards</h3>
	<img class="stretch" data-src="/images/talks/sarscov2_variant_rt_trajectories_2021_09_23.png">
	<div class="citation">
		Figgins and Bedford. Unpublished.
	</div>
</section>

<section>
	<h3>Consistent differences in variant-specific transmission rate across states</h3>
	<img class="stretch" data-src="/images/talks/sarscov2_variant_rt_growth_advantage_2021_09_23.png">
	<div class="citation">
		Figgins and Bedford. Unpublished.
	</div>
</section>

<section>
	<h3>Consistent reductions in variant-specific Rt from vaccination</h3>
	<img class="stretch" data-src="/images/talks/sarscov2_variant_rt_vaccination_2021_09_23.png">
	<div class="citation">
		Figgins and Bedford. Unpublished.
	</div>
</section>

<section class="left-align">
	<h3>
		Future work would ideally tie together granular empirical estimates of viral fitness from frequency data
		together with mutational and phenotypic predictors to learn what is driving variant success
	</h3>
</section>

<section>
	<h3>Model of additive effects of individual mutations shows little fitness variation within Delta</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_mutational_fitness_2021_11_12.png">
	<div class="citation">
		<a href="https://www.medrxiv.org/content/10.1101/2021.09.07.21263228v1">Obermeyer et al. 2021. medRxiv.</a>
	</div>
</section>

<!-- Potential for epistasis -->

<section data-transition="fade" data-background="#8EBC66">
	<h1 class="title">Potential for epistasis</h1>
</section>

<section>
	<h3>Mutations at spike 484 and 501 perhaps less fit on Delta background</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_484K_501Y_2021_10_19.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov/gisaid/global?c=gt-S_484,501">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>Mutations at spike 484 and 501 perhaps less fit on Delta background (zoomed to Delta)</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_484K_501Y_delta_2021_11_12.png">
	<div class="citation">
		<a href="https://nextstrain.org/groups/neherlab/ncov/21A.Delta?c=gt-S_484,501">nextstrain.org</a>
	</div>
</section>

<!-- Future perspectives -->

<section data-transition="fade" data-background="#AABD52">
	<h1 class="title">Future perspectives</h1>
</section>

<section class="left-align">
	<h3>Broad expectations based on comparison with flu</h3>
	<p>
	<ol>
		<li>
			R0 of Delta is perhaps ~5 compared to R0 of flu of ~2. At same rates of evolution and waning expect more COVID circulation.
		</li>
		<li>
			Rates of adaptive sequence evolution in SARS-CoV-2 have been about ~5X that of H3N2 flu.
			Expect this to slow down, but suggests that virus will be generally quite adaptable.
		</li>
		<li>
			IFR of COVID is roughly comparable to flu once you have prior immunity granting durable memory responses.
		</li>
	</ol>
	</p>
	<p>
		1, 2 and 3 suggest a virus that circulates like flu does at perhaps higher levels (given R0), but individual infections
		aren't much more severe in terms of mortality than H3N2 influenza. Expect perhaps 50-100K deaths in the US each year.
	</p>
</section>

<section class="left-align">
	<h3>SARS-CoV-2 scientific (and prosaic) priorities</h3>
	<p>
	<ol>
		<li>
			Forecasting evolution and ensuring vaccine updates match circulating viruses
		</li>
		<li>
			Universal vaccine to engender broader immune response and/or to target more conserved epitopes
		</li>
		<li>
			Plentiful testing, contact tracing and a public health apparatus
		</li>
		<li>
			Treatments, ventilation, etc...
		</li>
	</ol>
	</p>
</section>

<section>
	<h3>Acknowledgements</h3>

	<p class="even-smaller">
		<b>SARS-CoV-2 genomic epi</b>: Data producers from all over the world, GISAID and the Nextstrain team
	</p>

	<p class="tiny"><b>Bedford Lab</b>:
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/john-huddleston.jpg"> John Huddleston,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/james-hadfield.jpg"> James Hadfield,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/katie-kistler.jpg"> Katie Kistler,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/louise-moncla.jpg"> Louise Moncla,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/maya-lewinsohn.jpg"> Maya Lewinsohn,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/thomas-sibley.jpg"> Thomas Sibley,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/jover-lee.jpg"> Jover Lee,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/cassia-wagner.jpg"> Cassia Wagner,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/miguel-paredes.jpg"> Miguel Paredes,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/nicola-mueller.jpg"> Nicola Müller,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/marlin-figgins.jpg"> Marlin Figgins,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/eli-harkins.jpg"> Eli Harkins,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/denisse-sequeira.jpg"> Denisse Sequeira</span>
	</p>

	<div class="pull-right" style="vertical-align: bottom; display: table-cell;">
		<img width="11%" data-src="/images/talks/logo_fred_hutch_vertical.png">
		&nbsp; &nbsp;
		<img width="8%" data-src="/images/talks/logo_pew.png">
		&nbsp; &nbsp;
		<img width="14%" data-src="/images/talks/logo_bmgf.png">
		&nbsp; &nbsp;
		<img width="12%" data-src="/images/talks/logo_gates_ventures.png">
		&nbsp; &nbsp;
		<img width="11%" data-src="/images/talks/logo_wellcome_trust.jpg">
		&nbsp; &nbsp;
		<img width="7%" data-src="/images/talks/logo_nih.jpg">
		&nbsp; &nbsp;
		<img width="8%" data-src="/images/talks/logo_hhmi.jpg">
	</div>
</section>
