---
layout: paper
title: Quantifying within-host evolution of H5N1 influenza in humans and poultry in Cambodia
image: /images/papers/moncla-h5n1-cambodia.png
authors: Moncla LH, Bedford T, Dussart P, Horm SV, Rith S, Buchy P, Karlsson EA, Li L, Liu Y, Zhu H, Guan Y, Friedrich TC, Horwood PF.
year: 2020
ref: Moncla et al. 2020. PLoS Pathog.
journal: "PLoS Pathog 16: e1008191."
pdf: /pdfs/papers/moncla-h5n1-cambodia.pdf
doi: 10.1371/journal.ppat.1008191
github: https://github.com/blab/h5n1-cambodia/
---

# Abstract

Avian influenza viruses (AIVs) periodically cross species barriers and infect humans. The likelihood that an AIV will evolve mammalian transmissibility depends on acquiring and selecting mutations during spillover, but data from natural infection is limited. We analyze deep sequencing data from infected humans and domestic ducks in Cambodia to examine how H5N1 viruses evolve during spillover. Overall, viral populations in both species are predominated by low-frequency (<10%) variation shaped by purifying selection and genetic drift, and half of the variants detected within-host are never detected on the H5N1 virus phylogeny. However, we do detect a subset of mutations linked to human receptor binding and replication (PB2 E627K, HA A150V, and HA Q238L) that arose in multiple, independent humans. PB2 E627K and HA A150V were also enriched along phylogenetic branches leading to human infections, suggesting that they are likely human-adaptive. Our data show that H5N1 viruses generate putative human-adapting mutations during natural spillover infection, many of which are detected at >5% frequency within-host. However, short infection times, genetic drift, and purifying selection likely restrict their ability to evolve extensively during a single infection. Applying evolutionary methods to sequence data, we reveal a detailed view of H5N1 virus adaptive potential, and develop a foundation for studying host-adaptation in other zoonotic viruses.
