---
layout: paper
title: Specific allelic discrimination of N501Y and other SARS-CoV-2 mutations by ddPCR detects B.1.1.7 lineage in Washington State
image: /images/papers/perchetti-ddpcr-variant-detection.png
authors: Perchetti GA, Zhu H, Mills MG, Shrestha L, Wagner C, Mohamed Bakhash S, Lin M, Xie H, Meei-Li Huang, Mathias P, Bedford T, Jerome KR, Greninger AL, Roychoudhury P.
year: 2021
ref: Perchetti et al. 2021. J Med Virol.
journal: "J Med Virol 93: 5931-5941."
pdf: /pdfs/papers/perchetti-ddpcr-variant-detection.pdf
doi: 10.1002/jmv.27155
---

# Abstract

Real-time epidemiological tracking of variants of concern (VOCs) can help limit the spread of more contagious forms of SARS-CoV-2, such as those containing the N501Y mutation. Typically, genetic sequencing is required to be able to track VOCs in real-time. However, sequencing can take time and may not be accessible in all laboratories. Genotyping by RT-ddPCR offers an alternative to rapidly detect VOCs through discrimination of specific alleles such as N501Y which is associated with increased transmissibility and virulence. Here we describe the first cases of the B.1.1.7 lineage of SARS-CoV-2 detected in Washington State by using a combination of RT-PCR, RT-ddPCR, and next-generation sequencing. We initially screened 1,035 samples positive for SARS-CoV-2 by our CDC-based laboratory developed assay using ThermoFisher's multiplex RT-PCR COVID-19 assay over four weeks from late December 2020 to early January 2021. S gene target failures (SGTF) were subsequently assayed by RT-ddPCR to confirm four mutations within the S gene associated with the B.1.1.7 lineage: a deletion at amino acid (AA) 69-70 (ACATGT), deletion at AA 145, (TTA), N501Y mutation (TAT), and S982A mutation (GCA). All four targets were detected in two specimens; follow-up sequencing revealed a total of 9 mutations in the S gene and phylogenetic clustering within the B.1.1.7 lineage. Next, we continued screening samples for SGTF detecting 23 additional B.1.1.7 variants by RT-ddPCR and confirmed by sequencing. As VOCs become increasingly prevalent, molecular diagnostic tools like RT-ddPCR can be utilized to quickly, accurately, and sensitively distinguish more contagious lineages of SARS-CoV-2.
