---
layout: paper
title: Integration of genomic sequencing into the response to the Ebola virus outbreak in Nord Kivu, Democratic Republic of the Congo
image: /images/papers/kinganda-lusamaki-ebola-drc.png
authors: Kinganda-Lusamaki E, Black A, Mukadi DB, Hadfield J, Mbala-Kingebeni P, Pratt CB, Aziza A, Diagne MM, White B, Bisento N, Nsunda B, Akonga M, Faye M, Faye O, Edidi-Atani F, Matondo-Kuamfumu M, Mambu-Mbika F, Bulabula J, Di Paola N, Pauthner MG, Andersen KG, Palacios G, Delaporte E, Sall AA, Peeters M, Wiley MR, Ahuka-Mundeke S, Bedford T, Muyembe-Tamfum  J-J.
year: 2021
ref: Kinganda-Lusamaki et al. 2021. Nat Med.
journal: "Nat Med 27: 710-716."
pdf: /pdfs/papers/kinganda-lusamaki-ebola-drc.pdf
doi: 10.1038/s41591-021-01302-z
github: https://github.com/blab/ebola-narrative-ms/
---

# Abstract

On 1 August 2018, the Democratic Republic of the Congo (DRC) declared its tenth Ebola virus disease (EVD) outbreak. To aid the epidemiologic response, the Institut National de Recherche Biomédicale (INRB) implemented an end-to-end genomic surveillance system, including sequencing, bioinformatic analysis and dissemination of genomic epidemiologic results to frontline public health workers. We report 744 new genomes sampled between 27 July 2018 and 27 April 2020 generated by this surveillance effort. Together with previously available sequence data (n = 48 genomes), these data represent almost 24% of all laboratory-confirmed Ebola virus (EBOV) infections in DRC in the period analyzed. We inferred spatiotemporal transmission dynamics from the genomic data as new sequences were generated, and disseminated the results to support epidemiologic response efforts. Here we provide an overview of how this genomic surveillance system functioned, present a full phylodynamic analysis of 792 Ebola genomes from the Nord Kivu outbreak and discuss how the genomic surveillance data informed response efforts and public health decision making.
