---
layout: paper
title: Rapid and parallel adaptive mutations in spike S1 drive clade success in SARS-CoV-2
image: /images/papers/kistler-sarscov2-adaptive-evolution.png
authors: Kistler KE, Huddleston J, Bedford T.
year: 2022
ref: Kistler et al. 2022. Cell Host Microbe.
journal: "Cell Host Microbe 30: 545-555."
pdf: /pdfs/papers/kistler-sarscov2-adaptive-evolution.pdf
supplement: /pdfs/papers/kistler-sarscov2-adaptive-evolution-supp.pdf
doi: 10.1016/j.chom.2022.03.018
github: https://github.com/blab/sarscov2-adaptive-evolution
---

# Abstract

The SARS-CoV-2 pandemic has resulted in numerous virus variants, some of which have altered receptor-binding or antigenic phenotypes. Here, we quantify the degree to which adaptive evolution is driving the accumulation of mutations across the genome. We correlate clade growth with mutation accumulation, compare rates of nonsynonymous to synonymous divergence, assess temporal clustering of mutations and evaluate the evolutionary success of individual mutations. We find that spike S1 is the focus of adaptive evolution, but also identify positively-selected mutations in other genes (notably Nsp6) that are sculpting the evolutionary trajectory of SARS-CoV-2. Adaptive changes in S1 accumulated rapidly, resulting in a remarkably high ratio of nonsynonymous to synonymous divergence that is 2.5X greater than that observed in influenza hemagglutinin HA1 at the beginning of the 2009 H1N1 pandemic. These findings uncover a high degree of adaptation in S1 and suggest that SARS-CoV-2 may undergo antigenic drift.
